From 39b57e3e1ff38d5bdc4fc6c0434277395ba48619 Mon Sep 17 00:00:00 2001
From: Emil Renner Berthing <emil.renner.berthing@canonical.com>
Date: Thu, 29 Aug 2024 12:31:00 +0200
Subject: [PATCH 081/128] UBUNTU: [Config] Update for the ESWIN EIC7700

The ESWIN EIC7700 SoC has non-coherent peripheral DMAs and handles it in
a non-standard way, so unfortunately this kernel cannot be built to
support multiple SoCs.
Hence we disable support for other platforms and enable drivers for the
EIC7700.

Signed-off-by: Emil Renner Berthing <emil.renner.berthing@canonical.com>
---
 debian/config/annotations | 347 ++++++++------------------------------
 1 file changed, 66 insertions(+), 281 deletions(-)

diff --git a/debian/config/annotations b/debian/config/annotations
index 58cbc445b0f9..76581116f2b8 100644
--- a/debian/config/annotations
+++ b/debian/config/annotations
@@ -414,15 +414,6 @@ CONFIG_SERIAL_DEV_BUS                           note<'LP: #1739939'>
 CONFIG_SERIAL_DEV_CTRL_TTYPORT                  policy<{'riscv64': 'y'}>
 CONFIG_SERIAL_DEV_CTRL_TTYPORT                  note<'LP: #1739939'>
 
-CONFIG_SERIAL_SH_SCI                            policy<{'riscv64': 'y'}>
-CONFIG_SERIAL_SH_SCI                            note<'LP: #2022361'>
-
-CONFIG_SERIAL_SH_SCI_CONSOLE                    policy<{'riscv64': 'y'}>
-CONFIG_SERIAL_SH_SCI_CONSOLE                    note<'LP: #2022361'>
-
-CONFIG_SERIAL_SH_SCI_EARLYCON                   policy<{'riscv64': 'y'}>
-CONFIG_SERIAL_SH_SCI_EARLYCON                   note<'LP: #2022361'>
-
 CONFIG_SMC                                      policy<{'riscv64': 'm'}>
 CONFIG_SMC                                      note<'LP: #1789934'>
 
@@ -748,7 +739,7 @@ CONFIG_AF_RXRPC_IPV6                            policy<{'riscv64': 'y'}>
 CONFIG_AF_UNIX_OOB                              policy<{'riscv64': 'y'}>
 CONFIG_AHCI_CEVA                                policy<{'riscv64': 'm'}>
 CONFIG_AHCI_DWC                                 policy<{'riscv64': 'm'}>
-CONFIG_AHCI_SUNXI                               policy<{'riscv64': 'n'}>
+CONFIG_AHCI_ESWIN                               policy<{'riscv64': 'y'}>
 CONFIG_AIC79XX_CMDS_PER_DEVICE                  policy<{'riscv64': '32'}>
 CONFIG_AIC79XX_DEBUG_ENABLE                     policy<{'riscv64': 'n'}>
 CONFIG_AIC79XX_DEBUG_MASK                       policy<{'riscv64': '0'}>
@@ -780,7 +771,6 @@ CONFIG_ALTERA_TSE                               policy<{'riscv64': 'm'}>
 CONFIG_ALX                                      policy<{'riscv64': 'm'}>
 CONFIG_AL_FIC                                   policy<{'riscv64': 'y'}>
 CONFIG_AM2315                                   policy<{'riscv64': 'm'}>
-CONFIG_AMBA_PL08X                               policy<{'riscv64': 'y'}>
 CONFIG_AMD8111_ETH                              policy<{'riscv64': 'm'}>
 CONFIG_AMD_PHY                                  policy<{'riscv64': 'm'}>
 CONFIG_AMIGA_PARTITION                          policy<{'riscv64': 'y'}>
@@ -800,7 +790,6 @@ CONFIG_AQTION                                   policy<{'riscv64': 'm'}>
 CONFIG_AQUANTIA_PHY                             policy<{'riscv64': 'm'}>
 CONFIG_AR5523                                   policy<{'riscv64': 'm'}>
 CONFIG_ARCH_DMA_ADDR_T_64BIT                    policy<{'riscv64': 'y'}>
-CONFIG_ARCH_DMA_DEFAULT_COHERENT                policy<{'riscv64': 'y'}>
 CONFIG_ARCH_ENABLE_HUGEPAGE_MIGRATION           policy<{'riscv64': 'y'}>
 CONFIG_ARCH_ENABLE_SPLIT_PMD_PTLOCK             policy<{'riscv64': 'y'}>
 CONFIG_ARCH_ENABLE_THP_MIGRATION                policy<{'riscv64': 'y'}>
@@ -809,7 +798,9 @@ CONFIG_ARCH_HAS_CURRENT_STACK_POINTER           policy<{'riscv64': 'y'}>
 CONFIG_ARCH_HAS_DEBUG_VIRTUAL                   policy<{'riscv64': 'y'}>
 CONFIG_ARCH_HAS_DEBUG_VM_PGTABLE                policy<{'riscv64': 'y'}>
 CONFIG_ARCH_HAS_DEBUG_WX                        policy<{'riscv64': 'y'}>
+CONFIG_ARCH_HAS_DMA_CLEAR_UNCACHED              policy<{'riscv64': 'y'}>
 CONFIG_ARCH_HAS_DMA_PREP_COHERENT               policy<{'riscv64': 'y'}>
+CONFIG_ARCH_HAS_DMA_SET_UNCACHED                policy<{'riscv64': 'y'}>
 CONFIG_ARCH_HAS_ELF_RANDOMIZE                   policy<{'riscv64': 'y'}>
 CONFIG_ARCH_HAS_FORTIFY_SOURCE                  policy<{'riscv64': 'y'}>
 CONFIG_ARCH_HAS_GCOV_PROFILE_ALL                policy<{'riscv64': 'y'}>
@@ -827,17 +818,18 @@ CONFIG_ARCH_HAS_STRICT_MODULE_RWX               policy<{'riscv64': 'y'}>
 CONFIG_ARCH_HAS_SYNC_DMA_FOR_CPU                policy<{'riscv64': 'y'}>
 CONFIG_ARCH_HAS_SYNC_DMA_FOR_DEVICE             policy<{'riscv64': 'y'}>
 CONFIG_ARCH_HAS_SYSCALL_WRAPPER                 policy<{'riscv64': 'y'}>
+CONFIG_ARCH_HAS_TEARDOWN_DMA_OPS                policy<{'riscv64': 'y'}>
 CONFIG_ARCH_HAS_TICK_BROADCAST                  policy<{'riscv64': 'y'}>
 CONFIG_ARCH_HAS_UBSAN_SANITIZE_ALL              policy<{'riscv64': 'y'}>
 CONFIG_ARCH_HAS_VDSO_DATA                       policy<{'riscv64': 'y'}>
-CONFIG_ARCH_MICROCHIP_POLARFIRE                 policy<{'riscv64': 'y'}>
+CONFIG_ARCH_KEEP_MEMBLOCK                       policy<{'riscv64': 'y'}>
 CONFIG_ARCH_MMAP_RND_BITS_MAX                   policy<{'riscv64': '24'}>
 CONFIG_ARCH_MMAP_RND_BITS_MIN                   policy<{'riscv64': '18'}>
 CONFIG_ARCH_MMAP_RND_COMPAT_BITS_MAX            policy<{'riscv64': '17'}>
 CONFIG_ARCH_MMAP_RND_COMPAT_BITS_MIN            policy<{'riscv64': '8'}>
 CONFIG_ARCH_OPTIONAL_KERNEL_RWX                 policy<{'riscv64': 'y'}>
 CONFIG_ARCH_OPTIONAL_KERNEL_RWX_DEFAULT         policy<{'riscv64': 'y'}>
-CONFIG_ARCH_RENESAS                             policy<{'riscv64': 'y'}>
+CONFIG_ARCH_RENESAS                             policy<{'riscv64': 'n'}>
 CONFIG_ARCH_RV64I                               policy<{'riscv64': 'y'}>
 CONFIG_ARCH_SELECTS_KEXEC                       policy<{'riscv64': 'y'}>
 CONFIG_ARCH_SELECTS_KEXEC_FILE                  policy<{'riscv64': 'y'}>
@@ -845,8 +837,7 @@ CONFIG_ARCH_SELECT_MEMORY_MODEL                 policy<{'riscv64': 'y'}>
 CONFIG_ARCH_SIFIVE                              policy<{'riscv64': 'y'}>
 CONFIG_ARCH_SPARSEMEM_ENABLE                    policy<{'riscv64': 'y'}>
 CONFIG_ARCH_STACKWALK                           policy<{'riscv64': 'y'}>
-CONFIG_ARCH_STARFIVE                            policy<{'riscv64': 'y'}>
-CONFIG_ARCH_SUNXI                               policy<{'riscv64': 'y'}>
+CONFIG_ARCH_SUNXI                               policy<{'riscv64': 'n'}>
 CONFIG_ARCH_SUPPORTS_ACPI                       policy<{'riscv64': 'y'}>
 CONFIG_ARCH_SUPPORTS_ATOMIC_RMW                 policy<{'riscv64': 'y'}>
 CONFIG_ARCH_SUPPORTS_CFI_CLANG                  policy<{'riscv64': 'y'}>
@@ -862,7 +853,7 @@ CONFIG_ARCH_SUPPORTS_PAGE_TABLE_CHECK           policy<{'riscv64': 'y'}>
 CONFIG_ARCH_SUPPORTS_PER_VMA_LOCK               policy<{'riscv64': 'y'}>
 CONFIG_ARCH_SUPPORTS_UPROBES                    policy<{'riscv64': 'y'}>
 CONFIG_ARCH_SUSPEND_POSSIBLE                    policy<{'riscv64': 'y'}>
-CONFIG_ARCH_THEAD                               policy<{'riscv64': 'y'}>
+CONFIG_ARCH_THEAD                               policy<{'riscv64': 'n'}>
 CONFIG_ARCH_USE_MEMREMAP_PROT                   policy<{'riscv64': 'y'}>
 CONFIG_ARCH_USE_MEMTEST                         policy<{'riscv64': 'y'}>
 CONFIG_ARCH_USE_QUEUED_RWLOCKS                  policy<{'riscv64': 'y'}>
@@ -885,11 +876,8 @@ CONFIG_ARCNET_COM90xxIO                         policy<{'riscv64': 'm'}>
 CONFIG_ARCNET_RAW                               policy<{'riscv64': 'm'}>
 CONFIG_ARCNET_RIM_I                             policy<{'riscv64': 'm'}>
 CONFIG_ARCX_ANYBUS_CONTROLLER                   policy<{'riscv64': 'm'}>
-CONFIG_ARM_AMBA                                 policy<{'riscv64': 'y'}>
-CONFIG_ARM_MHU                                  policy<{'riscv64': 'm'}>
-CONFIG_ARM_MHU_V2                               policy<{'riscv64': 'm'}>
-CONFIG_ARM_PL172_MPMC                           policy<{'riscv64': 'm'}>
-CONFIG_ARM_SUN8I_A33_MBUS_DEVFREQ               policy<{'riscv64': 'm'}>
+CONFIG_ARM_SMMU_V3                              policy<{'riscv64': 'y'}>
+CONFIG_ARM_SMMU_V3_SVA                          policy<{'riscv64': 'n'}>
 CONFIG_AS3935                                   policy<{'riscv64': 'm'}>
 CONFIG_AS73211                                  policy<{'riscv64': 'm'}>
 CONFIG_ASM_MODVERSIONS                          policy<{'riscv64': 'y'}>
@@ -1468,15 +1456,12 @@ CONFIG_CAN_PEAK_PCIEFD                          policy<{'riscv64': 'm'}>
 CONFIG_CAN_PEAK_USB                             policy<{'riscv64': 'm'}>
 CONFIG_CAN_PLX_PCI                              policy<{'riscv64': 'm'}>
 CONFIG_CAN_RAW                                  policy<{'riscv64': 'm'}>
-CONFIG_CAN_RCAR                                 policy<{'riscv64': 'm'}>
-CONFIG_CAN_RCAR_CANFD                           policy<{'riscv64': 'm'}>
 CONFIG_CAN_RX_OFFLOAD                           policy<{'riscv64': 'y'}>
 CONFIG_CAN_SJA1000                              policy<{'riscv64': 'm'}>
 CONFIG_CAN_SJA1000_ISA                          policy<{'riscv64': 'm'}>
 CONFIG_CAN_SJA1000_PLATFORM                     policy<{'riscv64': 'm'}>
 CONFIG_CAN_SLCAN                                policy<{'riscv64': 'm'}>
 CONFIG_CAN_SOFTING                              policy<{'riscv64': 'm'}>
-CONFIG_CAN_SUN4I                                policy<{'riscv64': 'm'}>
 CONFIG_CAN_UCAN                                 policy<{'riscv64': 'm'}>
 CONFIG_CAN_VCAN                                 policy<{'riscv64': 'm'}>
 CONFIG_CAN_VXCAN                                policy<{'riscv64': 'm'}>
@@ -1635,20 +1620,9 @@ CONFIG_CIFS_SWN_UPCALL                          policy<{'riscv64': 'y'}>
 CONFIG_CIFS_UPCALL                              policy<{'riscv64': 'y'}>
 CONFIG_CIFS_XATTR                               policy<{'riscv64': 'y'}>
 CONFIG_CLANG_VERSION                            policy<{'riscv64': '0'}>
-CONFIG_CLKSRC_MMIO                              policy<{'riscv64': 'y'}>
 CONFIG_CLK_ANALOGBITS_WRPLL_CLN28HPC            policy<{'riscv64': 'y'}>
-CONFIG_CLK_RCAR_USB2_CLOCK_SEL                  policy<{'riscv64': 'y'}>
-CONFIG_CLK_RENESAS                              policy<{'riscv64': 'y'}>
 CONFIG_CLK_SIFIVE                               policy<{'riscv64': 'y'}>
 CONFIG_CLK_SIFIVE_PRCI                          policy<{'riscv64': 'y'}>
-CONFIG_CLK_STARFIVE_JH7100                      policy<{'riscv64': 'n'}>
-CONFIG_CLK_STARFIVE_JH7110_AON                  policy<{'riscv64': 'm'}>
-CONFIG_CLK_STARFIVE_JH7110_ISP                  policy<{'riscv64': 'm'}>
-CONFIG_CLK_STARFIVE_JH7110_PLL                  policy<{'riscv64': 'y'}>
-CONFIG_CLK_STARFIVE_JH7110_STG                  policy<{'riscv64': 'm'}>
-CONFIG_CLK_STARFIVE_JH7110_SYS                  policy<{'riscv64': 'y'}>
-CONFIG_CLK_STARFIVE_JH7110_VOUT                 policy<{'riscv64': 'm'}>
-CONFIG_CLK_STARFIVE_JH71X0                      policy<{'riscv64': 'y'}>
 CONFIG_CLK_TWL6040                              policy<{'riscv64': 'm'}>
 CONFIG_CLONE_BACKWARDS                          policy<{'riscv64': 'y'}>
 CONFIG_CLS_U32_MARK                             policy<{'riscv64': 'y'}>
@@ -1668,7 +1642,10 @@ CONFIG_CMA_SIZE_SEL_MBYTES                      policy<{'riscv64': 'y'}>
 CONFIG_CMA_SIZE_SEL_MIN                         policy<{'riscv64': 'n'}>
 CONFIG_CMA_SIZE_SEL_PERCENTAGE                  policy<{'riscv64': 'n'}>
 CONFIG_CMA_SYSFS                                policy<{'riscv64': 'y'}>
-CONFIG_CMDLINE                                  policy<{'riscv64': '""'}>
+CONFIG_CMDLINE                                  policy<{'riscv64': '"clk_ignore_unused"'}>
+CONFIG_CMDLINE_EXTEND                           policy<{'riscv64': 'y'}>
+CONFIG_CMDLINE_FALLBACK                         policy<{'riscv64': 'n'}>
+CONFIG_CMDLINE_FORCE                            policy<{'riscv64': 'n'}>
 CONFIG_CMDLINE_PARTITION                        policy<{'riscv64': 'y'}>
 CONFIG_CMODEL_MEDANY                            policy<{'riscv64': 'y'}>
 CONFIG_CMODEL_MEDLOW                            policy<{'riscv64': 'n'}>
@@ -1811,6 +1788,7 @@ CONFIG_COMMON_CLK_BD718XX                       policy<{'riscv64': 'm'}>
 CONFIG_COMMON_CLK_CDCE706                       policy<{'riscv64': 'm'}>
 CONFIG_COMMON_CLK_CDCE925                       policy<{'riscv64': 'm'}>
 CONFIG_COMMON_CLK_CS2000_CP                     policy<{'riscv64': 'm'}>
+CONFIG_COMMON_CLK_EIC7700                       policy<{'riscv64': 'y'}>
 CONFIG_COMMON_CLK_FIXED_MMIO                    policy<{'riscv64': 'y'}>
 CONFIG_COMMON_CLK_LOCHNAGAR                     policy<{'riscv64': 'm'}>
 CONFIG_COMMON_CLK_MAX77686                      policy<{'riscv64': 'm'}>
@@ -1935,7 +1913,6 @@ CONFIG_CRYPTO_CTS                               policy<{'riscv64': 'y'}>
 CONFIG_CRYPTO_CURVE25519                        policy<{'riscv64': 'm'}>
 CONFIG_CRYPTO_DEFLATE                           policy<{'riscv64': 'y'}>
 CONFIG_CRYPTO_DES                               policy<{'riscv64': 'm'}>
-CONFIG_CRYPTO_DEV_ALLWINNER                     policy<{'riscv64': 'y'}>
 CONFIG_CRYPTO_DEV_AMLOGIC_GXL                   policy<{'riscv64': 'm'}>
 CONFIG_CRYPTO_DEV_AMLOGIC_GXL_DEBUG             policy<{'riscv64': 'n'}>
 CONFIG_CRYPTO_DEV_ATMEL_ECC                     policy<{'riscv64': 'm'}>
@@ -1943,7 +1920,6 @@ CONFIG_CRYPTO_DEV_ATMEL_I2C                     policy<{'riscv64': 'm'}>
 CONFIG_CRYPTO_DEV_ATMEL_SHA204A                 policy<{'riscv64': 'm'}>
 CONFIG_CRYPTO_DEV_CCREE                         policy<{'riscv64': 'm'}>
 CONFIG_CRYPTO_DEV_CHELSIO                       policy<{'riscv64': 'm'}>
-CONFIG_CRYPTO_DEV_JH7110                        policy<{'riscv64': 'm'}>
 CONFIG_CRYPTO_DEV_NITROX                        policy<{'riscv64': 'm'}>
 CONFIG_CRYPTO_DEV_NITROX_CNN55XX                policy<{'riscv64': 'm'}>
 CONFIG_CRYPTO_DEV_QAT                           policy<{'riscv64': 'm'}>
@@ -1955,18 +1931,6 @@ CONFIG_CRYPTO_DEV_QAT_C62XVF                    policy<{'riscv64': 'm'}>
 CONFIG_CRYPTO_DEV_QAT_DH895xCC                  policy<{'riscv64': 'm'}>
 CONFIG_CRYPTO_DEV_QAT_DH895xCCVF                policy<{'riscv64': 'm'}>
 CONFIG_CRYPTO_DEV_SAFEXCEL                      policy<{'riscv64': 'm'}>
-CONFIG_CRYPTO_DEV_SUN4I_SS                      policy<{'riscv64': 'm'}>
-CONFIG_CRYPTO_DEV_SUN4I_SS_DEBUG                policy<{'riscv64': 'n'}>
-CONFIG_CRYPTO_DEV_SUN4I_SS_PRNG                 policy<{'riscv64': 'y'}>
-CONFIG_CRYPTO_DEV_SUN8I_CE                      policy<{'riscv64': 'm'}>
-CONFIG_CRYPTO_DEV_SUN8I_CE_DEBUG                policy<{'riscv64': 'n'}>
-CONFIG_CRYPTO_DEV_SUN8I_CE_HASH                 policy<{'riscv64': 'y'}>
-CONFIG_CRYPTO_DEV_SUN8I_CE_PRNG                 policy<{'riscv64': 'y'}>
-CONFIG_CRYPTO_DEV_SUN8I_CE_TRNG                 policy<{'riscv64': 'y'}>
-CONFIG_CRYPTO_DEV_SUN8I_SS                      policy<{'riscv64': 'm'}>
-CONFIG_CRYPTO_DEV_SUN8I_SS_DEBUG                policy<{'riscv64': 'n'}>
-CONFIG_CRYPTO_DEV_SUN8I_SS_HASH                 policy<{'riscv64': 'y'}>
-CONFIG_CRYPTO_DEV_SUN8I_SS_PRNG                 policy<{'riscv64': 'y'}>
 CONFIG_CRYPTO_DEV_VIRTIO                        policy<{'riscv64': 'm'}>
 CONFIG_CRYPTO_DH                                policy<{'riscv64': 'y'}>
 CONFIG_CRYPTO_DH_RFC7919_GROUPS                 policy<{'riscv64': 'y'}>
@@ -2221,7 +2185,6 @@ CONFIG_DMARD10                                  policy<{'riscv64': 'm'}>
 CONFIG_DMATEST                                  policy<{'riscv64': 'n'}>
 CONFIG_DMA_ACPI                                 policy<{'riscv64': 'y'}>
 CONFIG_DMA_API_DEBUG                            policy<{'riscv64': 'n'}>
-CONFIG_DMA_BOUNCE_UNALIGNED_KMALLOC             policy<{'riscv64': 'y'}>
 CONFIG_DMA_COHERENT_POOL                        policy<{'riscv64': 'y'}>
 CONFIG_DMA_DECLARE_COHERENT                     policy<{'riscv64': 'y'}>
 CONFIG_DMA_DIRECT_REMAP                         policy<{'riscv64': 'y'}>
@@ -2232,10 +2195,8 @@ CONFIG_DMA_NONCOHERENT_MMAP                     policy<{'riscv64': 'y'}>
 CONFIG_DMA_NUMA_CMA                             policy<{'riscv64': 'y'}>
 CONFIG_DMA_OF                                   policy<{'riscv64': 'y'}>
 CONFIG_DMA_OPS                                  policy<{'riscv64': 'y'}>
-CONFIG_DMA_RESTRICTED_POOL                      policy<{'riscv64': 'y'}>
 CONFIG_DMA_SHARED_BUFFER                        policy<{'riscv64': 'y'}>
-CONFIG_DMA_SUN6I                                policy<{'riscv64': 'm'}>
-CONFIG_DMA_VIRTUAL_CHANNELS                     policy<{'riscv64': 'y'}>
+CONFIG_DMA_VIRTUAL_CHANNELS                     policy<{'riscv64': 'm'}>
 CONFIG_DM_AUDIT                                 policy<{'riscv64': 'y'}>
 CONFIG_DM_BIO_PRISON                            policy<{'riscv64': 'm'}>
 CONFIG_DM_BUFIO                                 policy<{'riscv64': 'm'}>
@@ -2325,11 +2286,6 @@ CONFIG_DRM_DISPLAY_HELPER                       policy<{'riscv64': 'm'}>
 CONFIG_DRM_DP_AUX_BUS                           policy<{'riscv64': 'm'}>
 CONFIG_DRM_DP_AUX_CHARDEV                       policy<{'riscv64': 'y'}>
 CONFIG_DRM_DP_CEC                               policy<{'riscv64': 'y'}>
-CONFIG_DRM_DW_HDMI                              policy<{'riscv64': 'm'}>
-CONFIG_DRM_DW_HDMI_AHB_AUDIO                    policy<{'riscv64': 'm'}>
-CONFIG_DRM_DW_HDMI_CEC                          policy<{'riscv64': 'm'}>
-CONFIG_DRM_DW_HDMI_GP_AUDIO                     policy<{'riscv64': 'm'}>
-CONFIG_DRM_DW_HDMI_I2S_AUDIO                    policy<{'riscv64': 'm'}>
 CONFIG_DRM_ETNAVIV                              policy<{'riscv64': 'n'}>
 CONFIG_DRM_EXEC                                 policy<{'riscv64': 'm'}>
 CONFIG_DRM_FBDEV_EMULATION                      policy<{'riscv64': 'y'}>
@@ -2460,10 +2416,8 @@ CONFIG_DRM_PARADE_PS8640                        policy<{'riscv64': 'm'}>
 CONFIG_DRM_QXL                                  policy<{'riscv64': 'm'}>
 CONFIG_DRM_RADEON                               policy<{'riscv64': 'm'}>
 CONFIG_DRM_RADEON_USERPTR                       policy<{'riscv64': 'n'}>
-CONFIG_DRM_RZG2L_MIPI_DSI                       policy<{'riscv64': 'm'}>
 CONFIG_DRM_SAMSUNG_DSIM                         policy<{'riscv64': 'm'}>
 CONFIG_DRM_SCHED                                policy<{'riscv64': 'm'}>
-CONFIG_DRM_SHMOBILE                             policy<{'riscv64': 'm'}>
 CONFIG_DRM_SII902X                              policy<{'riscv64': 'm'}>
 CONFIG_DRM_SII9234                              policy<{'riscv64': 'm'}>
 CONFIG_DRM_SIL_SII8620                          policy<{'riscv64': 'm'}>
@@ -2473,11 +2427,6 @@ CONFIG_DRM_SSD130X                              policy<{'riscv64': 'm'}>
 CONFIG_DRM_SSD130X_I2C                          policy<{'riscv64': 'm'}>
 CONFIG_DRM_SSD130X_SPI                          policy<{'riscv64': 'm'}>
 CONFIG_DRM_SUBALLOC_HELPER                      policy<{'riscv64': 'm'}>
-CONFIG_DRM_SUN4I                                policy<{'riscv64': 'm'}>
-CONFIG_DRM_SUN6I_DSI                            policy<{'riscv64': 'm'}>
-CONFIG_DRM_SUN8I_DW_HDMI                        policy<{'riscv64': 'm'}>
-CONFIG_DRM_SUN8I_MIXER                          policy<{'riscv64': 'm'}>
-CONFIG_DRM_SUN8I_TCON_TOP                       policy<{'riscv64': 'm'}>
 CONFIG_DRM_THINE_THC63LVD1024                   policy<{'riscv64': 'm'}>
 CONFIG_DRM_TI_DLPC3433                          policy<{'riscv64': 'm'}>
 CONFIG_DRM_TI_SN65DSI83                         policy<{'riscv64': 'm'}>
@@ -2713,11 +2662,9 @@ CONFIG_DVB_ZL10353                              policy<{'riscv64': 'm'}>
 CONFIG_DWC_XLGMAC                               policy<{'riscv64': 'm'}>
 CONFIG_DWC_XLGMAC_PCI                           policy<{'riscv64': 'm'}>
 CONFIG_DWMAC_DWC_QOS_ETH                        policy<{'riscv64': 'm'}>
+CONFIG_DWMAC_EIC7700                            policy<{'riscv64': 'm'}>
 CONFIG_DWMAC_GENERIC                            policy<{'riscv64': 'm'}>
 CONFIG_DWMAC_INTEL_PLAT                         policy<{'riscv64': 'm'}>
-CONFIG_DWMAC_STARFIVE                           policy<{'riscv64': 'm'}>
-CONFIG_DWMAC_SUN8I                              policy<{'riscv64': 'm'}>
-CONFIG_DWMAC_SUNXI                              policy<{'riscv64': 'm'}>
 CONFIG_DW_AXI_DMAC                              policy<{'riscv64': 'm'}>
 CONFIG_DW_DMAC                                  policy<{'riscv64': 'm'}>
 CONFIG_DW_DMAC_CORE                             policy<{'riscv64': 'm'}>
@@ -2800,6 +2747,8 @@ CONFIG_ERRATA_THEAD                             policy<{'riscv64': 'y'}>
 CONFIG_ERRATA_THEAD_CMO                         policy<{'riscv64': 'y'}>
 CONFIG_ERRATA_THEAD_PBMT                        policy<{'riscv64': 'y'}>
 CONFIG_ERRATA_THEAD_PMU                         policy<{'riscv64': 'y'}>
+CONFIG_ESWIN_SND_ES8388_CODEC                   policy<{'riscv64': 'y'}>
+CONFIG_ESWIN_SND_SOC_CODECS                     policy<{'riscv64': 'y'}>
 CONFIG_ET131X                                   policy<{'riscv64': 'm'}>
 CONFIG_ETHERNET                                 policy<{'riscv64': 'y'}>
 CONFIG_ETHOC                                    policy<{'riscv64': 'm'}>
@@ -2932,7 +2881,6 @@ CONFIG_FB_S3_DDC                                policy<{'riscv64': 'y'}>
 CONFIG_FB_SAVAGE                                policy<{'riscv64': 'm'}>
 CONFIG_FB_SAVAGE_ACCEL                          policy<{'riscv64': 'n'}>
 CONFIG_FB_SAVAGE_I2C                            policy<{'riscv64': 'y'}>
-CONFIG_FB_SH_MOBILE_LCDC                        policy<{'riscv64': 'm'}>
 CONFIG_FB_SIMPLE                                policy<{'riscv64': 'm'}>
 CONFIG_FB_SIS                                   policy<{'riscv64': 'm'}>
 CONFIG_FB_SIS_300                               policy<{'riscv64': 'y'}>
@@ -3254,9 +3202,7 @@ CONFIG_GPIO_PCF857X                             policy<{'riscv64': 'm'}>
 CONFIG_GPIO_PCIE_IDIO_24                        policy<{'riscv64': 'm'}>
 CONFIG_GPIO_PCI_IDIO_16                         policy<{'riscv64': 'm'}>
 CONFIG_GPIO_PISOSR                              policy<{'riscv64': 'm'}>
-CONFIG_GPIO_PL061                               policy<{'riscv64': 'm'}>
 CONFIG_GPIO_RC5T583                             policy<{'riscv64': 'y'}>
-CONFIG_GPIO_RCAR                                policy<{'riscv64': 'm'}>
 CONFIG_GPIO_RDC321X                             policy<{'riscv64': 'm'}>
 CONFIG_GPIO_REGMAP                              policy<{'riscv64': 'm'}>
 CONFIG_GPIO_SIFIVE                              policy<{'riscv64': 'y'}>
@@ -3590,13 +3536,10 @@ CONFIG_HWMON                                    policy<{'riscv64': 'y'}>
 CONFIG_HWMON_DEBUG_CHIP                         policy<{'riscv64': 'n'}>
 CONFIG_HWMON_VID                                policy<{'riscv64': 'm'}>
 CONFIG_HWSPINLOCK                               policy<{'riscv64': 'y'}>
-CONFIG_HWSPINLOCK_SUN6I                         policy<{'riscv64': 'm'}>
 CONFIG_HW_CONSOLE                               policy<{'riscv64': 'y'}>
 CONFIG_HW_RANDOM                                policy<{'riscv64': 'y'}>
 CONFIG_HW_RANDOM_BA431                          policy<{'riscv64': 'm'}>
 CONFIG_HW_RANDOM_CCTRNG                         policy<{'riscv64': 'm'}>
-CONFIG_HW_RANDOM_JH7110                         policy<{'riscv64': 'm'}>
-CONFIG_HW_RANDOM_POLARFIRE_SOC                  policy<{'riscv64': 'm'}>
 CONFIG_HW_RANDOM_TIMERIOMEM                     policy<{'riscv64': 'm'}>
 CONFIG_HW_RANDOM_TPM                            policy<{'riscv64': 'y'}>
 CONFIG_HW_RANDOM_VIRTIO                         policy<{'riscv64': 'm'}>
@@ -3649,7 +3592,6 @@ CONFIG_I2C_HID_OF_GOODIX                        policy<{'riscv64': 'm'}>
 CONFIG_I2C_I801                                 policy<{'riscv64': 'm'}>
 CONFIG_I2C_ISCH                                 policy<{'riscv64': 'm'}>
 CONFIG_I2C_KEMPLD                               policy<{'riscv64': 'm'}>
-CONFIG_I2C_MICROCHIP_CORE                       policy<{'riscv64': 'm'}>
 CONFIG_I2C_MUX                                  policy<{'riscv64': 'm'}>
 CONFIG_I2C_MUX_GPIO                             policy<{'riscv64': 'm'}>
 CONFIG_I2C_MUX_GPMUX                            policy<{'riscv64': 'm'}>
@@ -3659,22 +3601,16 @@ CONFIG_I2C_MUX_PCA9541                          policy<{'riscv64': 'm'}>
 CONFIG_I2C_MUX_PCA954x                          policy<{'riscv64': 'm'}>
 CONFIG_I2C_MUX_PINCTRL                          policy<{'riscv64': 'm'}>
 CONFIG_I2C_MUX_REG                              policy<{'riscv64': 'm'}>
-CONFIG_I2C_MV64XXX                              policy<{'riscv64': 'm'}>
 CONFIG_I2C_NFORCE2                              policy<{'riscv64': 'm'}>
-CONFIG_I2C_NOMADIK                              policy<{'riscv64': 'm'}>
 CONFIG_I2C_NVIDIA_GPU                           policy<{'riscv64': 'm'}>
 CONFIG_I2C_OCORES                               policy<{'riscv64': 'm'}>
 CONFIG_I2C_PARPORT                              policy<{'riscv64': 'm'}>
 CONFIG_I2C_PCA_PLATFORM                         policy<{'riscv64': 'm'}>
 CONFIG_I2C_PCI1XXXX                             policy<{'riscv64': 'm'}>
 CONFIG_I2C_PIIX4                                policy<{'riscv64': 'm'}>
-CONFIG_I2C_RCAR                                 policy<{'riscv64': 'm'}>
-CONFIG_I2C_RIIC                                 policy<{'riscv64': 'm'}>
 CONFIG_I2C_RK3X                                 policy<{'riscv64': 'm'}>
 CONFIG_I2C_ROBOTFUZZ_OSIF                       policy<{'riscv64': 'm'}>
-CONFIG_I2C_RZV2M                                policy<{'riscv64': 'm'}>
 CONFIG_I2C_SCMI                                 policy<{'riscv64': 'm'}>
-CONFIG_I2C_SH_MOBILE                            policy<{'riscv64': 'm'}>
 CONFIG_I2C_SI470X                               policy<{'riscv64': 'm'}>
 CONFIG_I2C_SI4713                               policy<{'riscv64': 'm'}>
 CONFIG_I2C_SIMTEC                               policy<{'riscv64': 'm'}>
@@ -3852,6 +3788,7 @@ CONFIG_INFINIBAND_USER_MEM                      policy<{'riscv64': 'y'}>
 CONFIG_INFINIBAND_VIRT_DMA                      policy<{'riscv64': 'y'}>
 CONFIG_INFINIBAND_VMWARE_PVRDMA                 policy<{'riscv64': 'm'}>
 CONFIG_INFTL                                    policy<{'riscv64': 'm'}>
+CONFIG_INITRAMFS_FORCE                          policy<{'riscv64': 'n'}>
 CONFIG_INITRAMFS_PRESERVE_MTIME                 policy<{'riscv64': 'y'}>
 CONFIG_INITRAMFS_SOURCE                         policy<{'riscv64': '""'}>
 CONFIG_INIT_ENV_ARG_LIMIT                       policy<{'riscv64': '32'}>
@@ -3976,7 +3913,12 @@ CONFIG_IOMMU_API                                policy<{'riscv64': 'y'}>
 CONFIG_IOMMU_DEFAULT_DMA_LAZY                   policy<{'riscv64': 'n'}>
 CONFIG_IOMMU_DEFAULT_DMA_STRICT                 policy<{'riscv64': 'y'}>
 CONFIG_IOMMU_DEFAULT_PASSTHROUGH                policy<{'riscv64': 'n'}>
-CONFIG_IOMMU_IOVA                               policy<{'riscv64': 'm'}>
+CONFIG_IOMMU_DMA                                policy<{'riscv64': 'y'}>
+CONFIG_IOMMU_ESWIN                              policy<{'riscv64': 'y'}>
+CONFIG_IOMMU_IOVA                               policy<{'riscv64': 'y'}>
+CONFIG_IOMMU_IO_PGTABLE                         policy<{'riscv64': 'y'}>
+CONFIG_IOMMU_IO_PGTABLE_LPAE                    policy<{'riscv64': 'y'}>
+CONFIG_IOMMU_IO_PGTABLE_LPAE_SELFTEST           policy<{'riscv64': 'n'}>
 CONFIG_IOMMU_SUPPORT                            policy<{'riscv64': 'y'}>
 CONFIG_IONIC                                    policy<{'riscv64': 'm'}>
 CONFIG_IOSCHED_BFQ                              policy<{'riscv64': 'm'}>
@@ -4147,6 +4089,7 @@ CONFIG_IRQ_BYPASS_MANAGER                       policy<{'riscv64': 'm'}>
 CONFIG_IRQ_DOMAIN                               policy<{'riscv64': 'y'}>
 CONFIG_IRQ_DOMAIN_HIERARCHY                     policy<{'riscv64': 'y'}>
 CONFIG_IRQ_FORCED_THREADING                     policy<{'riscv64': 'y'}>
+CONFIG_IRQ_MSI_IOMMU                            policy<{'riscv64': 'y'}>
 CONFIG_IRQ_POLL                                 policy<{'riscv64': 'y'}>
 CONFIG_IRQ_SIM                                  policy<{'riscv64': 'y'}>
 CONFIG_IRQ_STACKS                               policy<{'riscv64': 'y'}>
@@ -4181,7 +4124,6 @@ CONFIG_IR_SHARP_DECODER                         policy<{'riscv64': 'm'}>
 CONFIG_IR_SONY_DECODER                          policy<{'riscv64': 'm'}>
 CONFIG_IR_SPI                                   policy<{'riscv64': 'm'}>
 CONFIG_IR_STREAMZAP                             policy<{'riscv64': 'm'}>
-CONFIG_IR_SUNXI                                 policy<{'riscv64': 'm'}>
 CONFIG_IR_TOY                                   policy<{'riscv64': 'm'}>
 CONFIG_IR_TTUSBIR                               policy<{'riscv64': 'm'}>
 CONFIG_IR_XMP_DECODER                           policy<{'riscv64': 'm'}>
@@ -4242,7 +4184,6 @@ CONFIG_JFS_FS                                   policy<{'riscv64': 'm'}>
 CONFIG_JFS_POSIX_ACL                            policy<{'riscv64': 'y'}>
 CONFIG_JFS_SECURITY                             policy<{'riscv64': 'y'}>
 CONFIG_JFS_STATISTICS                           policy<{'riscv64': 'y'}>
-CONFIG_JH71XX_PMU                               policy<{'riscv64': 'y'}>
 CONFIG_JME                                      policy<{'riscv64': 'm'}>
 CONFIG_JOLIET                                   policy<{'riscv64': 'y'}>
 CONFIG_JOYSTICK_A3D                             policy<{'riscv64': 'm'}>
@@ -4332,7 +4273,6 @@ CONFIG_KEYBOARD_QT2160                          policy<{'riscv64': 'm'}>
 CONFIG_KEYBOARD_SAMSUNG                         policy<{'riscv64': 'm'}>
 CONFIG_KEYBOARD_STMPE                           policy<{'riscv64': 'm'}>
 CONFIG_KEYBOARD_STOWAWAY                        policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_SUN4I_LRADC                     policy<{'riscv64': 'm'}>
 CONFIG_KEYBOARD_SUNKBD                          policy<{'riscv64': 'm'}>
 CONFIG_KEYBOARD_TC3589X                         policy<{'riscv64': 'm'}>
 CONFIG_KEYBOARD_TCA6416                         policy<{'riscv64': 'm'}>
@@ -4673,7 +4613,6 @@ CONFIG_MC3230                                   policy<{'riscv64': 'm'}>
 CONFIG_MCB                                      policy<{'riscv64': 'm'}>
 CONFIG_MCB_LPC                                  policy<{'riscv64': 'm'}>
 CONFIG_MCB_PCI                                  policy<{'riscv64': 'm'}>
-CONFIG_MCHP_CLK_MPFS                            policy<{'riscv64': 'y'}>
 CONFIG_MCP320X                                  policy<{'riscv64': 'm'}>
 CONFIG_MCP3422                                  policy<{'riscv64': 'm'}>
 CONFIG_MCP3911                                  policy<{'riscv64': 'm'}>
@@ -4709,7 +4648,6 @@ CONFIG_MDIO_MSCC_MIIM                           policy<{'riscv64': 'm'}>
 CONFIG_MDIO_MVUSB                               policy<{'riscv64': 'm'}>
 CONFIG_MDIO_OCTEON                              policy<{'riscv64': 'm'}>
 CONFIG_MDIO_REGMAP                              policy<{'riscv64': 'm'}>
-CONFIG_MDIO_SUN4I                               policy<{'riscv64': 'n'}>
 CONFIG_MDIO_THUNDER                             policy<{'riscv64': 'm'}>
 CONFIG_MD_AUTODETECT                            policy<{'riscv64': 'y'}>
 CONFIG_MD_BITMAP_FILE                           policy<{'riscv64': 'y'}>
@@ -4815,7 +4753,6 @@ CONFIG_MFD_88PM800                              policy<{'riscv64': 'm'}>
 CONFIG_MFD_88PM805                              policy<{'riscv64': 'm'}>
 CONFIG_MFD_88PM860X                             policy<{'riscv64': 'y'}>
 CONFIG_MFD_AAT2870_CORE                         policy<{'riscv64': 'y'}>
-CONFIG_MFD_AC100                                policy<{'riscv64': 'n'}>
 CONFIG_MFD_ACT8945A                             policy<{'riscv64': 'm'}>
 CONFIG_MFD_ARIZONA                              policy<{'riscv64': 'm'}>
 CONFIG_MFD_ARIZONA_I2C                          policy<{'riscv64': 'm'}>
@@ -4828,7 +4765,6 @@ CONFIG_MFD_ATMEL_FLEXCOM                        policy<{'riscv64': 'm'}>
 CONFIG_MFD_ATMEL_HLCDC                          policy<{'riscv64': 'm'}>
 CONFIG_MFD_AXP20X                               policy<{'riscv64': 'm'}>
 CONFIG_MFD_AXP20X_I2C                           policy<{'riscv64': 'm'}>
-CONFIG_MFD_AXP20X_RSB                           policy<{'riscv64': 'm'}>
 CONFIG_MFD_BCM590XX                             policy<{'riscv64': 'm'}>
 CONFIG_MFD_BD9571MWV                            policy<{'riscv64': 'm'}>
 CONFIG_MFD_CORE                                 policy<{'riscv64': 'y'}>
@@ -4915,8 +4851,6 @@ CONFIG_MFD_SMPRO                                policy<{'riscv64': 'm'}>
 CONFIG_MFD_STMFX                                policy<{'riscv64': 'm'}>
 CONFIG_MFD_STMPE                                policy<{'riscv64': 'y'}>
 CONFIG_MFD_STPMIC1                              policy<{'riscv64': 'm'}>
-CONFIG_MFD_SUN4I_GPADC                          policy<{'riscv64': 'm'}>
-CONFIG_MFD_SUN6I_PRCM                           policy<{'riscv64': 'y'}>
 CONFIG_MFD_SY7636A                              policy<{'riscv64': 'm'}>
 CONFIG_MFD_SYSCON                               policy<{'riscv64': 'y'}>
 CONFIG_MFD_TC3589X                              policy<{'riscv64': 'y'}>
@@ -5042,7 +4976,6 @@ CONFIG_MMA9553                                  policy<{'riscv64': 'm'}>
 CONFIG_MMC                                      policy<{'riscv64': 'y'}>
 CONFIG_MMC35240                                 policy<{'riscv64': 'm'}>
 CONFIG_MMC_ALCOR                                policy<{'riscv64': 'm'}>
-CONFIG_MMC_ARMMMCI                              policy<{'riscv64': 'm'}>
 CONFIG_MMC_BLOCK_MINORS                         policy<{'riscv64': '8'}>
 CONFIG_MMC_CB710                                policy<{'riscv64': 'm'}>
 CONFIG_MMC_CQHCI                                policy<{'riscv64': 'm'}>
@@ -5055,7 +4988,6 @@ CONFIG_MMC_DW_HI3798CV200                       policy<{'riscv64': 'm'}>
 CONFIG_MMC_DW_K3                                policy<{'riscv64': 'm'}>
 CONFIG_MMC_DW_PCI                               policy<{'riscv64': 'm'}>
 CONFIG_MMC_DW_PLTFM                             policy<{'riscv64': 'm'}>
-CONFIG_MMC_DW_STARFIVE                          policy<{'riscv64': 'm'}>
 CONFIG_MMC_HSQ                                  policy<{'riscv64': 'm'}>
 CONFIG_MMC_LITEX                                policy<{'riscv64': 'm'}>
 CONFIG_MMC_MTK                                  policy<{'riscv64': 'm'}>
@@ -5070,17 +5002,13 @@ CONFIG_MMC_SDHCI_MILBEAUT                       policy<{'riscv64': 'm'}>
 CONFIG_MMC_SDHCI_OF_ARASAN                      policy<{'riscv64': 'm'}>
 CONFIG_MMC_SDHCI_OF_AT91                        policy<{'riscv64': 'm'}>
 CONFIG_MMC_SDHCI_OF_DWCMSHC                     policy<{'riscv64': 'm'}>
+CONFIG_MMC_SDHCI_OF_ESWIN                       policy<{'riscv64': 'm'}>
+CONFIG_MMC_SDHCI_OF_SDIO_ESWIN                  policy<{'riscv64': 'm'}>
 CONFIG_MMC_SDHCI_PCI                            policy<{'riscv64': 'm'}>
 CONFIG_MMC_SDHCI_XENON                          policy<{'riscv64': 'm'}>
-CONFIG_MMC_SDHI                                 policy<{'riscv64': 'm'}>
-CONFIG_MMC_SDHI_SYS_DMAC                        policy<{'riscv64': 'm'}>
-CONFIG_MMC_SH_MMCIF                             policy<{'riscv64': 'm'}>
 CONFIG_MMC_SPI                                  policy<{'riscv64': 'y'}>
-CONFIG_MMC_STM32_SDMMC                          policy<{'riscv64': 'y'}>
-CONFIG_MMC_SUNXI                                policy<{'riscv64': 'm'}>
 CONFIG_MMC_TEST                                 policy<{'riscv64': 'n'}>
 CONFIG_MMC_TIFM_SD                              policy<{'riscv64': 'm'}>
-CONFIG_MMC_TMIO_CORE                            policy<{'riscv64': 'm'}>
 CONFIG_MMC_TOSHIBA_PCI                          policy<{'riscv64': 'm'}>
 CONFIG_MMC_USDHI6ROL0                           policy<{'riscv64': 'm'}>
 CONFIG_MMC_USHC                                 policy<{'riscv64': 'm'}>
@@ -5266,9 +5194,7 @@ CONFIG_MTD_NAND_INTEL_LGM                       policy<{'riscv64': 'm'}>
 CONFIG_MTD_NAND_MXIC                            policy<{'riscv64': 'm'}>
 CONFIG_MTD_NAND_NANDSIM                         policy<{'riscv64': 'm'}>
 CONFIG_MTD_NAND_PLATFORM                        policy<{'riscv64': 'm'}>
-CONFIG_MTD_NAND_RENESAS                         policy<{'riscv64': 'm'}>
 CONFIG_MTD_NAND_RICOH                           policy<{'riscv64': 'm'}>
-CONFIG_MTD_NAND_SUNXI                           policy<{'riscv64': 'n'}>
 CONFIG_MTD_ONENAND                              policy<{'riscv64': 'm'}>
 CONFIG_MTD_ONENAND_2X_PROGRAM                   policy<{'riscv64': 'y'}>
 CONFIG_MTD_ONENAND_GENERIC                      policy<{'riscv64': 'm'}>
@@ -5342,6 +5268,7 @@ CONFIG_ND_CLAIM                                 policy<{'riscv64': 'y'}>
 CONFIG_NE2K_PCI                                 policy<{'riscv64': 'm'}>
 CONFIG_NEED_DMA_MAP_STATE                       policy<{'riscv64': 'y'}>
 CONFIG_NEED_PER_CPU_EMBED_FIRST_CHUNK           policy<{'riscv64': 'y'}>
+CONFIG_NEED_SG_DMA_LENGTH                       policy<{'riscv64': 'y'}>
 CONFIG_NET                                      policy<{'riscv64': 'y'}>
 CONFIG_NETCONSOLE                               policy<{'riscv64': 'm'}>
 CONFIG_NETCONSOLE_DYNAMIC                       policy<{'riscv64': 'y'}>
@@ -5512,11 +5439,7 @@ CONFIG_NET_DSA_MV88E6XXX                        policy<{'riscv64': 'm'}>
 CONFIG_NET_DSA_MV88E6XXX_PTP                    policy<{'riscv64': 'y'}>
 CONFIG_NET_DSA_QCA8K                            policy<{'riscv64': 'm'}>
 CONFIG_NET_DSA_QCA8K_LEDS_SUPPORT               policy<{'riscv64': 'y'}>
-CONFIG_NET_DSA_REALTEK                          policy<{'riscv64': 'm'}>
-CONFIG_NET_DSA_REALTEK_MDIO                     policy<{'riscv64': 'm'}>
-CONFIG_NET_DSA_REALTEK_RTL8365MB                policy<{'riscv64': 'm'}>
-CONFIG_NET_DSA_REALTEK_RTL8366RB                policy<{'riscv64': 'm'}>
-CONFIG_NET_DSA_REALTEK_SMI                      policy<{'riscv64': 'm'}>
+CONFIG_NET_DSA_REALTEK                          policy<{'riscv64': 'n'}>
 CONFIG_NET_DSA_SJA1105                          policy<{'riscv64': 'm'}>
 CONFIG_NET_DSA_SJA1105_PTP                      policy<{'riscv64': 'y'}>
 CONFIG_NET_DSA_SJA1105_TAS                      policy<{'riscv64': 'y'}>
@@ -5642,7 +5565,6 @@ CONFIG_NET_VENDOR_ADAPTEC                       policy<{'riscv64': 'y'}>
 CONFIG_NET_VENDOR_ADI                           policy<{'riscv64': 'y'}>
 CONFIG_NET_VENDOR_AGERE                         policy<{'riscv64': 'y'}>
 CONFIG_NET_VENDOR_ALACRITECH                    policy<{'riscv64': 'y'}>
-CONFIG_NET_VENDOR_ALLWINNER                     policy<{'riscv64': 'y'}>
 CONFIG_NET_VENDOR_ALTEON                        policy<{'riscv64': 'y'}>
 CONFIG_NET_VENDOR_AMAZON                        policy<{'riscv64': 'y'}>
 CONFIG_NET_VENDOR_AMD                           policy<{'riscv64': 'y'}>
@@ -5988,7 +5910,6 @@ CONFIG_NVMEM_RAVE_SP_EEPROM                     policy<{'riscv64': 'm'}>
 CONFIG_NVMEM_REBOOT_MODE                        policy<{'riscv64': 'm'}>
 CONFIG_NVMEM_RMEM                               policy<{'riscv64': 'm'}>
 CONFIG_NVMEM_SPMI_SDAM                          policy<{'riscv64': 'm'}>
-CONFIG_NVMEM_SUNXI_SID                          policy<{'riscv64': 'm'}>
 CONFIG_NVMEM_SYSFS                              policy<{'riscv64': 'y'}>
 CONFIG_NVMEM_U_BOOT_ENV                         policy<{'riscv64': 'm'}>
 CONFIG_NVME_AUTH                                policy<{'riscv64': 'y'}>
@@ -6195,12 +6116,11 @@ CONFIG_PCIE_DW_PLAT_EP                          policy<{'riscv64': 'y'}>
 CONFIG_PCIE_DW_PLAT_HOST                        policy<{'riscv64': 'y'}>
 CONFIG_PCIE_ECRC                                policy<{'riscv64': 'n'}>
 CONFIG_PCIE_EDR                                 policy<{'riscv64': 'y'}>
+CONFIG_PCIE_ESWIN                               policy<{'riscv64': 'y'}>
 CONFIG_PCIE_FU740                               policy<{'riscv64': 'y'}>
 CONFIG_PCIE_MICROCHIP_HOST                      policy<{'riscv64': 'y'}>
 CONFIG_PCIE_PME                                 policy<{'riscv64': 'y'}>
 CONFIG_PCIE_PTM                                 policy<{'riscv64': 'y'}>
-CONFIG_PCIE_RCAR_EP                             policy<{'riscv64': 'y'}>
-CONFIG_PCIE_RCAR_HOST                           policy<{'riscv64': 'y'}>
 CONFIG_PCIE_XILINX                              policy<{'riscv64': 'y'}>
 CONFIG_PCIPCWATCHDOG                            policy<{'riscv64': 'm'}>
 CONFIG_PCI_ATMEL                                policy<{'riscv64': 'm'}>
@@ -6272,19 +6192,7 @@ CONFIG_PHY_PXA_28NM_HSIC                        policy<{'riscv64': 'm'}>
 CONFIG_PHY_PXA_28NM_USB2                        policy<{'riscv64': 'm'}>
 CONFIG_PHY_QCOM_USB_HS                          policy<{'riscv64': 'm'}>
 CONFIG_PHY_QCOM_USB_HSIC                        policy<{'riscv64': 'm'}>
-CONFIG_PHY_R8A779F0_ETHERNET_SERDES             policy<{'riscv64': 'm'}>
-CONFIG_PHY_RCAR_GEN2                            policy<{'riscv64': 'm'}>
-CONFIG_PHY_RCAR_GEN3_PCIE                       policy<{'riscv64': 'm'}>
-CONFIG_PHY_RCAR_GEN3_USB2                       policy<{'riscv64': 'm'}>
-CONFIG_PHY_RCAR_GEN3_USB3                       policy<{'riscv64': 'm'}>
 CONFIG_PHY_SAMSUNG_USB2                         policy<{'riscv64': 'm'}>
-CONFIG_PHY_STARFIVE_JH7110_DPHY_RX              policy<{'riscv64': 'm'}>
-CONFIG_PHY_STARFIVE_JH7110_PCIE                 policy<{'riscv64': 'm'}>
-CONFIG_PHY_STARFIVE_JH7110_USB                  policy<{'riscv64': 'm'}>
-CONFIG_PHY_SUN4I_USB                            policy<{'riscv64': 'm'}>
-CONFIG_PHY_SUN50I_USB3                          policy<{'riscv64': 'm'}>
-CONFIG_PHY_SUN6I_MIPI_DPHY                      policy<{'riscv64': 'm'}>
-CONFIG_PHY_SUN9I_USB                            policy<{'riscv64': 'n'}>
 CONFIG_PHY_TUSB1210                             policy<{'riscv64': 'm'}>
 CONFIG_PI433                                    policy<{'riscv64': 'm'}>
 CONFIG_PID_NS                                   policy<{'riscv64': 'y'}>
@@ -6301,6 +6209,7 @@ CONFIG_PINCTRL_CS47L90                          policy<{'riscv64': 'y'}>
 CONFIG_PINCTRL_CS47L92                          policy<{'riscv64': 'y'}>
 CONFIG_PINCTRL_CY8C95X0                         policy<{'riscv64': 'm'}>
 CONFIG_PINCTRL_DA9062                           policy<{'riscv64': 'm'}>
+CONFIG_PINCTRL_EIC7700                          policy<{'riscv64': 'y'}>
 CONFIG_PINCTRL_LOCHNAGAR                        policy<{'riscv64': 'm'}>
 CONFIG_PINCTRL_MADERA                           policy<{'riscv64': 'm'}>
 CONFIG_PINCTRL_MAX77620                         policy<{'riscv64': 'm'}>
@@ -6310,47 +6219,15 @@ CONFIG_PINCTRL_MCP23S08_SPI                     policy<{'riscv64': 'm'}>
 CONFIG_PINCTRL_MICROCHIP_SGPIO                  policy<{'riscv64': 'y'}>
 CONFIG_PINCTRL_OCELOT                           policy<{'riscv64': 'y'}>
 CONFIG_PINCTRL_PALMAS                           policy<{'riscv64': 'y'}>
-CONFIG_PINCTRL_RENESAS                          policy<{'riscv64': 'y'}>
 CONFIG_PINCTRL_RK805                            policy<{'riscv64': 'm'}>
 CONFIG_PINCTRL_SINGLE                           policy<{'riscv64': 'y'}>
-CONFIG_PINCTRL_STARFIVE_JH7100                  policy<{'riscv64': 'n'}>
-CONFIG_PINCTRL_STARFIVE_JH7110                  policy<{'riscv64': 'y'}>
-CONFIG_PINCTRL_STARFIVE_JH7110_AON              policy<{'riscv64': 'm'}>
-CONFIG_PINCTRL_STARFIVE_JH7110_SYS              policy<{'riscv64': 'y'}>
 CONFIG_PINCTRL_STMFX                            policy<{'riscv64': 'm'}>
-CONFIG_PINCTRL_SUN20I_D1                        policy<{'riscv64': 'y'}>
-CONFIG_PINCTRL_SUN4I_A10                        policy<{'riscv64': 'n'}>
-CONFIG_PINCTRL_SUN50I_A100                      policy<{'riscv64': 'n'}>
-CONFIG_PINCTRL_SUN50I_A100_R                    policy<{'riscv64': 'n'}>
-CONFIG_PINCTRL_SUN50I_A64                       policy<{'riscv64': 'n'}>
-CONFIG_PINCTRL_SUN50I_A64_R                     policy<{'riscv64': 'n'}>
-CONFIG_PINCTRL_SUN50I_H5                        policy<{'riscv64': 'n'}>
-CONFIG_PINCTRL_SUN50I_H6                        policy<{'riscv64': 'n'}>
-CONFIG_PINCTRL_SUN50I_H616                      policy<{'riscv64': 'n'}>
-CONFIG_PINCTRL_SUN50I_H616_R                    policy<{'riscv64': 'n'}>
-CONFIG_PINCTRL_SUN50I_H6_R                      policy<{'riscv64': 'n'}>
-CONFIG_PINCTRL_SUN5I                            policy<{'riscv64': 'n'}>
-CONFIG_PINCTRL_SUN6I_A31                        policy<{'riscv64': 'n'}>
-CONFIG_PINCTRL_SUN6I_A31_R                      policy<{'riscv64': 'n'}>
-CONFIG_PINCTRL_SUN8I_A23                        policy<{'riscv64': 'n'}>
-CONFIG_PINCTRL_SUN8I_A23_R                      policy<{'riscv64': 'n'}>
-CONFIG_PINCTRL_SUN8I_A33                        policy<{'riscv64': 'n'}>
-CONFIG_PINCTRL_SUN8I_A83T                       policy<{'riscv64': 'n'}>
-CONFIG_PINCTRL_SUN8I_A83T_R                     policy<{'riscv64': 'n'}>
-CONFIG_PINCTRL_SUN8I_H3                         policy<{'riscv64': 'n'}>
-CONFIG_PINCTRL_SUN8I_H3_R                       policy<{'riscv64': 'n'}>
-CONFIG_PINCTRL_SUN8I_V3S                        policy<{'riscv64': 'n'}>
-CONFIG_PINCTRL_SUN9I_A80                        policy<{'riscv64': 'n'}>
-CONFIG_PINCTRL_SUN9I_A80_R                      policy<{'riscv64': 'n'}>
-CONFIG_PINCTRL_SUNXI                            policy<{'riscv64': 'y'}>
 CONFIG_PINCTRL_SX150X                           policy<{'riscv64': 'y'}>
 CONFIG_PING                                     policy<{'riscv64': 'm'}>
 CONFIG_PINMUX                                   policy<{'riscv64': 'y'}>
 CONFIG_PKCS7_MESSAGE_PARSER                     policy<{'riscv64': 'y'}>
 CONFIG_PKCS7_TEST_KEY                           policy<{'riscv64': 'm'}>
 CONFIG_PKCS8_PRIVATE_KEY_PARSER                 policy<{'riscv64': 'm'}>
-CONFIG_PL320_MBOX                               policy<{'riscv64': 'y'}>
-CONFIG_PL330_DMA                                policy<{'riscv64': 'm'}>
 CONFIG_PLATFORM_MHU                             policy<{'riscv64': 'm'}>
 CONFIG_PLATFORM_SI4713                          policy<{'riscv64': 'm'}>
 CONFIG_PLAYSTATION_FF                           policy<{'riscv64': 'y'}>
@@ -6390,8 +6267,6 @@ CONFIG_PNFS_FLEXFILE_LAYOUT                     policy<{'riscv64': 'm'}>
 CONFIG_PNP                                      policy<{'riscv64': 'y'}>
 CONFIG_PNPACPI                                  policy<{'riscv64': 'y'}>
 CONFIG_PNP_DEBUG_MESSAGES                       policy<{'riscv64': 'n'}>
-CONFIG_POLARFIRE_SOC_MAILBOX                    policy<{'riscv64': 'm'}>
-CONFIG_POLARFIRE_SOC_SYS_CTRL                   policy<{'riscv64': 'm'}>
 CONFIG_POLYNOMIAL                               policy<{'riscv64': 'm'}>
 CONFIG_PORTABLE                                 policy<{'riscv64': 'y'}>
 CONFIG_POSIX_CPU_TIMERS_TASK_WORK               policy<{'riscv64': 'y'}>
@@ -6507,17 +6382,14 @@ CONFIG_PWM_ATMEL_TCB                            policy<{'riscv64': 'm'}>
 CONFIG_PWM_CLK                                  policy<{'riscv64': 'm'}>
 CONFIG_PWM_DEBUG                                policy<{'riscv64': 'n'}>
 CONFIG_PWM_DWC                                  policy<{'riscv64': 'm'}>
+CONFIG_PWM_ESWIN                                policy<{'riscv64': 'm'}>
 CONFIG_PWM_FSL_FTM                              policy<{'riscv64': 'm'}>
 CONFIG_PWM_IQS620A                              policy<{'riscv64': 'm'}>
 CONFIG_PWM_LP3943                               policy<{'riscv64': 'm'}>
-CONFIG_PWM_MICROCHIP_CORE                       policy<{'riscv64': 'm'}>
 CONFIG_PWM_NTXEC                                policy<{'riscv64': 'm'}>
 CONFIG_PWM_PCA9685                              policy<{'riscv64': 'm'}>
-CONFIG_PWM_RCAR                                 policy<{'riscv64': 'm'}>
-CONFIG_PWM_RENESAS_TPU                          policy<{'riscv64': 'm'}>
 CONFIG_PWM_SIFIVE                               policy<{'riscv64': 'm'}>
 CONFIG_PWM_STMPE                                policy<{'riscv64': 'y'}>
-CONFIG_PWM_SUN4I                                policy<{'riscv64': 'm'}>
 CONFIG_PWM_SYSFS                                policy<{'riscv64': 'y'}>
 CONFIG_PWM_TWL                                  policy<{'riscv64': 'm'}>
 CONFIG_PWM_TWL_LED                              policy<{'riscv64': 'm'}>
@@ -6573,7 +6445,7 @@ CONFIG_QUOTA_DEBUG                              policy<{'riscv64': 'n'}>
 CONFIG_QUOTA_NETLINK_INTERFACE                  policy<{'riscv64': 'y'}>
 CONFIG_QUOTA_TREE                               policy<{'riscv64': 'm'}>
 CONFIG_R6040                                    policy<{'riscv64': 'm'}>
-CONFIG_R8169                                    policy<{'riscv64': 'm'}>
+CONFIG_R8169                                    policy<{'riscv64': 'n'}>
 CONFIG_R8712U                                   policy<{'riscv64': 'm'}>
 CONFIG_RADIO_ADAPTERS                           policy<{'riscv64': 'm'}>
 CONFIG_RADIO_MAXIRADIO                          policy<{'riscv64': 'm'}>
@@ -6609,13 +6481,9 @@ CONFIG_RAPIDIO_RXS_GEN3                         policy<{'riscv64': 'm'}>
 CONFIG_RAPIDIO_TSI721                           policy<{'riscv64': 'm'}>
 CONFIG_RAS                                      policy<{'riscv64': 'y'}>
 CONFIG_RATIONAL                                 policy<{'riscv64': 'y'}>
-CONFIG_RAVB                                     policy<{'riscv64': 'm'}>
 CONFIG_RAVE_SP_CORE                             policy<{'riscv64': 'm'}>
 CONFIG_RAVE_SP_WATCHDOG                         policy<{'riscv64': 'm'}>
 CONFIG_RBTREE_TEST                              policy<{'riscv64': 'n'}>
-CONFIG_RCAR_DMAC                                policy<{'riscv64': 'm'}>
-CONFIG_RCAR_GEN3_THERMAL                        policy<{'riscv64': 'm'}>
-CONFIG_RCAR_THERMAL                             policy<{'riscv64': 'm'}>
 CONFIG_RCU_CPU_STALL_CPUTIME                    policy<{'riscv64': 'y'}>
 CONFIG_RCU_CPU_STALL_TIMEOUT                    policy<{'riscv64': '60'}>
 CONFIG_RCU_EQS_DEBUG                            policy<{'riscv64': 'n'}>
@@ -6650,7 +6518,7 @@ CONFIG_RD_ZSTD                                  policy<{'riscv64': 'y'}>
 CONFIG_READABLE_ASM                             policy<{'riscv64': 'n'}>
 CONFIG_READ_ONLY_THP_FOR_FS                     policy<{'riscv64': 'n'}>
 CONFIG_REALTEK_AUTOPM                           policy<{'riscv64': 'y'}>
-CONFIG_REALTEK_PHY                              policy<{'riscv64': 'm'}>
+CONFIG_REALTEK_PHY                              policy<{'riscv64': 'n'}>
 CONFIG_REBOOT_MODE                              policy<{'riscv64': 'm'}>
 CONFIG_REED_SOLOMON                             policy<{'riscv64': 'm'}>
 CONFIG_REED_SOLOMON_DEC16                       policy<{'riscv64': 'y'}>
@@ -6839,24 +6707,11 @@ CONFIG_RELAY                                    policy<{'riscv64': 'y'}>
 CONFIG_RELOCATABLE                              policy<{'riscv64': 'y'}>
 CONFIG_REMOTEPROC                               policy<{'riscv64': 'n'}>
 CONFIG_REMOTE_TARGET                            policy<{'riscv64': 'm'}>
-CONFIG_RENESAS_DMA                              policy<{'riscv64': 'y'}>
-CONFIG_RENESAS_ETHER_SWITCH                     policy<{'riscv64': 'm'}>
-CONFIG_RENESAS_OSTM                             policy<{'riscv64': 'y'}>
 CONFIG_RENESAS_PHY                              policy<{'riscv64': 'm'}>
-CONFIG_RENESAS_RPCIF                            policy<{'riscv64': 'm'}>
-CONFIG_RENESAS_RZAWDT                           policy<{'riscv64': 'm'}>
-CONFIG_RENESAS_RZG2LWDT                         policy<{'riscv64': 'm'}>
-CONFIG_RENESAS_RZN1WDT                          policy<{'riscv64': 'm'}>
-CONFIG_RENESAS_USB_DMAC                         policy<{'riscv64': 'm'}>
-CONFIG_RENESAS_WDT                              policy<{'riscv64': 'm'}>
 CONFIG_RESET_ATTACK_MITIGATION                  policy<{'riscv64': 'y'}>
 CONFIG_RESET_CONTROLLER                         policy<{'riscv64': 'y'}>
-CONFIG_RESET_POLARFIRE_SOC                      policy<{'riscv64': 'y'}>
+CONFIG_RESET_ESWIN_EIC7700                      policy<{'riscv64': 'y'}>
 CONFIG_RESET_SIMPLE                             policy<{'riscv64': 'y'}>
-CONFIG_RESET_STARFIVE_JH7100                    policy<{'riscv64': 'n'}>
-CONFIG_RESET_STARFIVE_JH7110                    policy<{'riscv64': 'y'}>
-CONFIG_RESET_STARFIVE_JH71X0                    policy<{'riscv64': 'y'}>
-CONFIG_RESET_SUNXI                              policy<{'riscv64': 'y'}>
 CONFIG_RESET_TI_SYSCON                          policy<{'riscv64': 'm'}>
 CONFIG_RESET_TI_TPS380X                         policy<{'riscv64': 'm'}>
 CONFIG_RETHOOK                                  policy<{'riscv64': 'y'}>
@@ -6880,8 +6735,17 @@ CONFIG_RISCV                                    policy<{'riscv64': 'y'}>
 CONFIG_RISCV_ALTERNATIVE                        policy<{'riscv64': 'y'}>
 CONFIG_RISCV_ALTERNATIVE_EARLY                  policy<{'riscv64': 'y'}>
 CONFIG_RISCV_BOOT_SPINWAIT                      policy<{'riscv64': 'n'}>
+CONFIG_RISCV_DIE0_CACHED_OFFSET                 policy<{'riscv64': '0x80000000'}>
+CONFIG_RISCV_DIE0_MEM_MAX_SIZE                  policy<{'riscv64': '0x800000000'}>
+CONFIG_RISCV_DIE0_UNCACHED_OFFSET               policy<{'riscv64': '0xc000000000'}>
+CONFIG_RISCV_DIE1_CACHED_OFFSET                 policy<{'riscv64': '0x2000000000'}>
+CONFIG_RISCV_DIE1_MEM_MAX_SIZE                  policy<{'riscv64': '0x800000000'}>
+CONFIG_RISCV_DIE1_UNCACHED_OFFSET               policy<{'riscv64': '0xe000000000'}>
 CONFIG_RISCV_DMA_NONCOHERENT                    policy<{'riscv64': 'y'}>
 CONFIG_RISCV_INTC                               policy<{'riscv64': 'y'}>
+CONFIG_RISCV_INTERLEAVE_CACHED_OFFSET           policy<{'riscv64': '0x4000000000'}>
+CONFIG_RISCV_INTERLEAVE_MEM_MAX_SIZE            policy<{'riscv64': '0x1000000000'}>
+CONFIG_RISCV_INTERLEAVE_UNCACHED_OFFSET         policy<{'riscv64': '0x10000000000'}>
 CONFIG_RISCV_ISA_C                              policy<{'riscv64': 'y'}>
 CONFIG_RISCV_ISA_FALLBACK                       policy<{'riscv64': 'y'}>
 CONFIG_RISCV_ISA_SVNAPOT                        policy<{'riscv64': 'y'}>
@@ -6891,6 +6755,7 @@ CONFIG_RISCV_ISA_V_DEFAULT_ENABLE               policy<{'riscv64': 'y'}>
 CONFIG_RISCV_ISA_ZBB                            policy<{'riscv64': 'y'}>
 CONFIG_RISCV_ISA_ZICBOM                         policy<{'riscv64': 'y'}>
 CONFIG_RISCV_ISA_ZICBOZ                         policy<{'riscv64': 'y'}>
+CONFIG_RISCV_NONSTANDARD_CACHE_OPS              policy<{'riscv64': 'y'}>
 CONFIG_RISCV_PMU                                policy<{'riscv64': 'y'}>
 CONFIG_RISCV_PMU_LEGACY                         policy<{'riscv64': 'y'}>
 CONFIG_RISCV_PMU_SBI                            policy<{'riscv64': 'y'}>
@@ -7015,6 +6880,7 @@ CONFIG_RTC_DRV_DS2404                           policy<{'riscv64': 'm'}>
 CONFIG_RTC_DRV_DS3232                           policy<{'riscv64': 'm'}>
 CONFIG_RTC_DRV_DS3232_HWMON                     policy<{'riscv64': 'y'}>
 CONFIG_RTC_DRV_EM3027                           policy<{'riscv64': 'm'}>
+CONFIG_RTC_DRV_ESWIN                            policy<{'riscv64': 'y'}>
 CONFIG_RTC_DRV_FM3130                           policy<{'riscv64': 'm'}>
 CONFIG_RTC_DRV_FTRTC010                         policy<{'riscv64': 'm'}>
 CONFIG_RTC_DRV_GOLDFISH                         policy<{'riscv64': 'y'}>
@@ -7055,9 +6921,6 @@ CONFIG_RTC_DRV_PCF8523                          policy<{'riscv64': 'm'}>
 CONFIG_RTC_DRV_PCF85363                         policy<{'riscv64': 'm'}>
 CONFIG_RTC_DRV_PCF8563                          policy<{'riscv64': 'm'}>
 CONFIG_RTC_DRV_PCF8583                          policy<{'riscv64': 'm'}>
-CONFIG_RTC_DRV_PL030                            policy<{'riscv64': 'm'}>
-CONFIG_RTC_DRV_PL031                            policy<{'riscv64': 'm'}>
-CONFIG_RTC_DRV_POLARFIRE_SOC                    policy<{'riscv64': 'm'}>
 CONFIG_RTC_DRV_R7301                            policy<{'riscv64': 'm'}>
 CONFIG_RTC_DRV_R9701                            policy<{'riscv64': 'm'}>
 CONFIG_RTC_DRV_RC5T583                          policy<{'riscv64': 'm'}>
@@ -7079,9 +6942,7 @@ CONFIG_RTC_DRV_RX8581                           policy<{'riscv64': 'm'}>
 CONFIG_RTC_DRV_S35390A                          policy<{'riscv64': 'm'}>
 CONFIG_RTC_DRV_S5M                              policy<{'riscv64': 'm'}>
 CONFIG_RTC_DRV_SD3078                           policy<{'riscv64': 'm'}>
-CONFIG_RTC_DRV_SH                               policy<{'riscv64': 'm'}>
 CONFIG_RTC_DRV_STK17TA8                         policy<{'riscv64': 'm'}>
-CONFIG_RTC_DRV_SUN6I                            policy<{'riscv64': 'y'}>
 CONFIG_RTC_DRV_TEST                             policy<{'riscv64': 'n'}>
 CONFIG_RTC_DRV_TPS6586X                         policy<{'riscv64': 'm'}>
 CONFIG_RTC_DRV_TPS65910                         policy<{'riscv64': 'm'}>
@@ -7169,6 +7030,7 @@ CONFIG_RTW89_DEBUGMSG                           policy<{'riscv64': 'y'}>
 CONFIG_RTW89_PCI                                policy<{'riscv64': 'm'}>
 CONFIG_RT_MUTEXES                               policy<{'riscv64': 'y'}>
 CONFIG_RUNTIME_TESTING_MENU                     policy<{'riscv64': 'y'}>
+CONFIG_RUST_IS_AVAILABLE                        policy<{'riscv64': 'y'}>
 CONFIG_RV                                       policy<{'riscv64': 'y'}>
 CONFIG_RV_MON_WWNR                              policy<{'riscv64': 'y'}>
 CONFIG_RV_REACTORS                              policy<{'riscv64': 'y'}>
@@ -7177,7 +7039,6 @@ CONFIG_RV_REACT_PRINTK                          policy<{'riscv64': 'y'}>
 CONFIG_RWSEM_SPIN_ON_OWNER                      policy<{'riscv64': 'y'}>
 CONFIG_RXKAD                                    policy<{'riscv64': 'y'}>
 CONFIG_RXPERF                                   policy<{'riscv64': 'm'}>
-CONFIG_RZG2L_THERMAL                            policy<{'riscv64': 'm'}>
 CONFIG_S2IO                                     policy<{'riscv64': 'm'}>
 CONFIG_SAMPLES                                  policy<{'riscv64': 'y'}>
 CONFIG_SAMPLE_AUXDISPLAY                        policy<{'riscv64': 'n'}>
@@ -7207,7 +7068,6 @@ CONFIG_SATA_NV                                  policy<{'riscv64': 'm'}>
 CONFIG_SATA_PMP                                 policy<{'riscv64': 'y'}>
 CONFIG_SATA_PROMISE                             policy<{'riscv64': 'm'}>
 CONFIG_SATA_QSTOR                               policy<{'riscv64': 'm'}>
-CONFIG_SATA_RCAR                                policy<{'riscv64': 'm'}>
 CONFIG_SATA_SIL                                 policy<{'riscv64': 'm'}>
 CONFIG_SATA_SIL24                               policy<{'riscv64': 'm'}>
 CONFIG_SATA_SIS                                 policy<{'riscv64': 'm'}>
@@ -7331,7 +7191,6 @@ CONFIG_SCSI_UFS_CRYPTO                          policy<{'riscv64': 'y'}>
 CONFIG_SCSI_UFS_DWC_TC_PCI                      policy<{'riscv64': 'm'}>
 CONFIG_SCSI_UFS_DWC_TC_PLATFORM                 policy<{'riscv64': 'm'}>
 CONFIG_SCSI_UFS_HWMON                           policy<{'riscv64': 'n'}>
-CONFIG_SCSI_UFS_RENESAS                         policy<{'riscv64': 'm'}>
 CONFIG_SCSI_WD719X                              policy<{'riscv64': 'm'}>
 CONFIG_SCTP_COOKIE_HMAC_MD5                     policy<{'riscv64': 'y'}>
 CONFIG_SCTP_COOKIE_HMAC_SHA1                    policy<{'riscv64': 'y'}>
@@ -7439,6 +7298,8 @@ CONFIG_SENSORS_EMC1403                          policy<{'riscv64': 'm'}>
 CONFIG_SENSORS_EMC2103                          policy<{'riscv64': 'm'}>
 CONFIG_SENSORS_EMC2305                          policy<{'riscv64': 'm'}>
 CONFIG_SENSORS_EMC6W201                         policy<{'riscv64': 'm'}>
+CONFIG_SENSORS_ESWIN_FAN_CONTROL                policy<{'riscv64': 'y'}>
+CONFIG_SENSORS_ESWIN_PVT                        policy<{'riscv64': 'y'}>
 CONFIG_SENSORS_F71805F                          policy<{'riscv64': 'm'}>
 CONFIG_SENSORS_F71882FG                         policy<{'riscv64': 'm'}>
 CONFIG_SENSORS_F75375S                          policy<{'riscv64': 'm'}>
@@ -7561,6 +7422,7 @@ CONFIG_SENSORS_NZXT_KRAKEN2                     policy<{'riscv64': 'm'}>
 CONFIG_SENSORS_NZXT_SMART2                      policy<{'riscv64': 'm'}>
 CONFIG_SENSORS_OCC_P8_I2C                       policy<{'riscv64': 'n'}>
 CONFIG_SENSORS_OCC_P9_SBE                       policy<{'riscv64': 'n'}>
+CONFIG_SENSORS_PAC1934                          policy<{'riscv64': 'm'}>
 CONFIG_SENSORS_PC87360                          policy<{'riscv64': 'm'}>
 CONFIG_SENSORS_PC87427                          policy<{'riscv64': 'm'}>
 CONFIG_SENSORS_PCF8591                          policy<{'riscv64': 'm'}>
@@ -7584,7 +7446,6 @@ CONFIG_SENSORS_SBTSI                            policy<{'riscv64': 'm'}>
 CONFIG_SENSORS_SCH5627                          policy<{'riscv64': 'm'}>
 CONFIG_SENSORS_SCH5636                          policy<{'riscv64': 'm'}>
 CONFIG_SENSORS_SCH56XX_COMMON                   policy<{'riscv64': 'm'}>
-CONFIG_SENSORS_SFCTEMP                          policy<{'riscv64': 'm'}>
 CONFIG_SENSORS_SHT15                            policy<{'riscv64': 'm'}>
 CONFIG_SENSORS_SHT21                            policy<{'riscv64': 'm'}>
 CONFIG_SENSORS_SHT3x                            policy<{'riscv64': 'm'}>
@@ -7647,7 +7508,6 @@ CONFIG_SERIAL_8250_DETECT_IRQ                   policy<{'riscv64': 'n'}>
 CONFIG_SERIAL_8250_DFL                          policy<{'riscv64': 'm'}>
 CONFIG_SERIAL_8250_DMA                          policy<{'riscv64': 'y'}>
 CONFIG_SERIAL_8250_DWLIB                        policy<{'riscv64': 'y'}>
-CONFIG_SERIAL_8250_EM                           policy<{'riscv64': 'n'}>
 CONFIG_SERIAL_8250_EXAR                         policy<{'riscv64': 'm'}>
 CONFIG_SERIAL_8250_EXTENDED                     policy<{'riscv64': 'y'}>
 CONFIG_SERIAL_8250_FINTEK                       policy<{'riscv64': 'y'}>
@@ -7667,14 +7527,13 @@ CONFIG_SERIAL_ALTERA_JTAGUART                   policy<{'riscv64': 'm'}>
 CONFIG_SERIAL_ALTERA_UART                       policy<{'riscv64': 'm'}>
 CONFIG_SERIAL_ALTERA_UART_BAUDRATE              policy<{'riscv64': '115200'}>
 CONFIG_SERIAL_ALTERA_UART_MAXPORTS              policy<{'riscv64': '4'}>
-CONFIG_SERIAL_AMBA_PL010                        policy<{'riscv64': 'm'}>
-CONFIG_SERIAL_AMBA_PL011                        policy<{'riscv64': 'm'}>
 CONFIG_SERIAL_ARC                               policy<{'riscv64': 'm'}>
 CONFIG_SERIAL_ARC_NR_PORTS                      policy<{'riscv64': '1'}>
 CONFIG_SERIAL_CONEXANT_DIGICOLOR                policy<{'riscv64': 'm'}>
 CONFIG_SERIAL_CORE                              policy<{'riscv64': 'y'}>
 CONFIG_SERIAL_CORE_CONSOLE                      policy<{'riscv64': 'y'}>
 CONFIG_SERIAL_EARLYCON                          policy<{'riscv64': 'y'}>
+CONFIG_SERIAL_EARLYCON_RISCV_SBI                policy<{'riscv64': 'n'}>
 CONFIG_SERIAL_EARLYCON_SEMIHOST                 policy<{'riscv64': 'y'}>
 CONFIG_SERIAL_FSL_LINFLEXUART                   policy<{'riscv64': 'm'}>
 CONFIG_SERIAL_FSL_LPUART                        policy<{'riscv64': 'm'}>
@@ -7697,8 +7556,6 @@ CONFIG_SERIAL_SC16IS7XX_I2C                     policy<{'riscv64': 'y'}>
 CONFIG_SERIAL_SC16IS7XX_SPI                     policy<{'riscv64': 'y'}>
 CONFIG_SERIAL_SCCNXP                            policy<{'riscv64': 'y'}>
 CONFIG_SERIAL_SCCNXP_CONSOLE                    policy<{'riscv64': 'y'}>
-CONFIG_SERIAL_SH_SCI_DMA                        policy<{'riscv64': 'y'}>
-CONFIG_SERIAL_SH_SCI_NR_UARTS                   policy<{'riscv64': '2'}>
 CONFIG_SERIAL_SIFIVE                            policy<{'riscv64': 'y'}>
 CONFIG_SERIAL_SIFIVE_CONSOLE                    policy<{'riscv64': 'y'}>
 CONFIG_SERIAL_SPRD                              policy<{'riscv64': 'm'}>
@@ -7707,7 +7564,6 @@ CONFIG_SERIAL_UARTLITE_NR_UARTS                 policy<{'riscv64': '1'}>
 CONFIG_SERIAL_XILINX_PS_UART                    policy<{'riscv64': 'm'}>
 CONFIG_SERIO                                    policy<{'riscv64': 'y'}>
 CONFIG_SERIO_ALTERA_PS2                         policy<{'riscv64': 'm'}>
-CONFIG_SERIO_AMBAKMI                            policy<{'riscv64': 'm'}>
 CONFIG_SERIO_APBPS2                             policy<{'riscv64': 'm'}>
 CONFIG_SERIO_ARC_PS2                            policy<{'riscv64': 'm'}>
 CONFIG_SERIO_GPIO_PS2                           policy<{'riscv64': 'm'}>
@@ -7717,7 +7573,6 @@ CONFIG_SERIO_PCIPS2                             policy<{'riscv64': 'm'}>
 CONFIG_SERIO_PS2MULT                            policy<{'riscv64': 'm'}>
 CONFIG_SERIO_RAW                                policy<{'riscv64': 'm'}>
 CONFIG_SERIO_SERPORT                            policy<{'riscv64': 'm'}>
-CONFIG_SERIO_SUN4I_PS2                          policy<{'riscv64': 'n'}>
 CONFIG_SFC                                      policy<{'riscv64': 'm'}>
 CONFIG_SFC_FALCON                               policy<{'riscv64': 'm'}>
 CONFIG_SFC_FALCON_MTD                           policy<{'riscv64': 'y'}>
@@ -7739,7 +7594,6 @@ CONFIG_SG_POOL                                  policy<{'riscv64': 'y'}>
 CONFIG_SHMEM                                    policy<{'riscv64': 'y'}>
 CONFIG_SHRINKER_DEBUG                           policy<{'riscv64': 'n'}>
 CONFIG_SHUFFLE_PAGE_ALLOCATOR                   policy<{'riscv64': 'y'}>
-CONFIG_SH_ETH                                   policy<{'riscv64': 'm'}>
 CONFIG_SI1133                                   policy<{'riscv64': 'm'}>
 CONFIG_SI1145                                   policy<{'riscv64': 'm'}>
 CONFIG_SI7005                                   policy<{'riscv64': 'm'}>
@@ -7840,6 +7694,7 @@ CONFIG_SND_DYNAMIC_MINORS                       policy<{'riscv64': 'y'}>
 CONFIG_SND_ECHO3G                               policy<{'riscv64': 'm'}>
 CONFIG_SND_ENS1370                              policy<{'riscv64': 'm'}>
 CONFIG_SND_ENS1371                              policy<{'riscv64': 'm'}>
+CONFIG_SND_ESWIN_DW_I2S                         policy<{'riscv64': 'm'}>
 CONFIG_SND_FIREFACE                             policy<{'riscv64': 'm'}>
 CONFIG_SND_FIREWIRE                             policy<{'riscv64': 'y'}>
 CONFIG_SND_FIREWIRE_DIGI00X                     policy<{'riscv64': 'm'}>
@@ -8079,7 +7934,6 @@ CONFIG_SND_SOC_IMG_SPDIF_IN                     policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_IMG_SPDIF_OUT                    policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_IMX_AUDMUX                       policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_INNO_RK3036                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_JH7110_TDM                       policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_LOCHNAGAR_SC                     policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_LPASS_MACRO_COMMON               policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_LPASS_RX_MACRO                   policy<{'riscv64': 'm'}>
@@ -8135,7 +7989,6 @@ CONFIG_SND_SOC_PCM512x                          policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_PCM512x_I2C                      policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_PCM512x_SPI                      policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_PEB2466                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_RCAR                             policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_RK3328                           policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_RK817                            policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_RL6231                           policy<{'riscv64': 'm'}>
@@ -8165,7 +8018,6 @@ CONFIG_SND_SOC_RT722_SDCA_SDW                   policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_RT9120                           policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_SDW_MOCKUP                       policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_SGTL5000                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_SH4_FSI                          policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_SI476X                           policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_SIGMADSP                         policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_SIGMADSP_I2C                     policy<{'riscv64': 'm'}>
@@ -8189,7 +8041,6 @@ CONFIG_SND_SOC_SSM3515                          policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_SSM4567                          policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_STA32X                           policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_STA350                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_STARFIVE                         policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_STI_SAS                          policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_TAS2552                          policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_TAS2562                          policy<{'riscv64': 'm'}>
@@ -8268,10 +8119,6 @@ CONFIG_SND_SOC_XILINX_SPDIF                     policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_XTFPGA_I2S                       policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_ZL38060                          policy<{'riscv64': 'm'}>
 CONFIG_SND_SPI                                  policy<{'riscv64': 'y'}>
-CONFIG_SND_SUN4I_CODEC                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SUN4I_I2S                            policy<{'riscv64': 'm'}>
-CONFIG_SND_SUN4I_SPDIF                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SUN50I_DMIC                          policy<{'riscv64': 'm'}>
 CONFIG_SND_SUPPORT_OLD_API                      policy<{'riscv64': 'y'}>
 CONFIG_SND_TEST_COMPONENT                       policy<{'riscv64': 'm'}>
 CONFIG_SND_TIMER                                policy<{'riscv64': 'm'}>
@@ -8307,10 +8154,10 @@ CONFIG_SOCK_CGROUP_DATA                         policy<{'riscv64': 'y'}>
 CONFIG_SOCK_RX_QUEUE_MAPPING                    policy<{'riscv64': 'y'}>
 CONFIG_SOCK_VALIDATE_XMIT                       policy<{'riscv64': 'y'}>
 CONFIG_SOC_BUS                                  policy<{'riscv64': 'y'}>
-CONFIG_SOC_MICROCHIP_POLARFIRE                  policy<{'riscv64': 'y'}>
-CONFIG_SOC_RENESAS                              policy<{'riscv64': 'y'}>
+CONFIG_SOC_MICROCHIP_POLARFIRE                  policy<{'riscv64': 'n'}>
 CONFIG_SOC_SIFIVE                               policy<{'riscv64': 'y'}>
-CONFIG_SOC_STARFIVE                             policy<{'riscv64': 'y'}>
+CONFIG_SOC_SIFIVE_EIC7700                       policy<{'riscv64': 'y'}>
+CONFIG_SOC_STARFIVE                             policy<{'riscv64': 'n'}>
 CONFIG_SOC_TI                                   policy<{'riscv64': 'y'}>
 CONFIG_SOC_VIRT                                 policy<{'riscv64': 'y'}>
 CONFIG_SOFTIRQ_ON_OWN_STACK                     policy<{'riscv64': 'y'}>
@@ -8364,6 +8211,7 @@ CONFIG_SPI_DW_DMA                               policy<{'riscv64': 'y'}>
 CONFIG_SPI_DW_MMIO                              policy<{'riscv64': 'm'}>
 CONFIG_SPI_DW_PCI                               policy<{'riscv64': 'm'}>
 CONFIG_SPI_DYNAMIC                              policy<{'riscv64': 'y'}>
+CONFIG_SPI_ESWIN_BOOTSPI                        policy<{'riscv64': 'm'}>
 CONFIG_SPI_FSI                                  policy<{'riscv64': 'm'}>
 CONFIG_SPI_FSL_LIB                              policy<{'riscv64': 'y'}>
 CONFIG_SPI_FSL_SPI                              policy<{'riscv64': 'y'}>
@@ -8378,23 +8226,15 @@ CONFIG_SPI_MUX                                  policy<{'riscv64': 'm'}>
 CONFIG_SPI_MXIC                                 policy<{'riscv64': 'm'}>
 CONFIG_SPI_OC_TINY                              policy<{'riscv64': 'm'}>
 CONFIG_SPI_PCI1XXXX                             policy<{'riscv64': 'm'}>
-CONFIG_SPI_PL022                                policy<{'riscv64': 'm'}>
 CONFIG_SPI_PXA2XX                               policy<{'riscv64': 'm'}>
 CONFIG_SPI_PXA2XX_PCI                           policy<{'riscv64': 'm'}>
-CONFIG_SPI_RPCIF                                policy<{'riscv64': 'm'}>
-CONFIG_SPI_RSPI                                 policy<{'riscv64': 'm'}>
-CONFIG_SPI_RZV2M_CSI                            policy<{'riscv64': 'm'}>
 CONFIG_SPI_SC18IS602                            policy<{'riscv64': 'm'}>
-CONFIG_SPI_SH_HSPI                              policy<{'riscv64': 'm'}>
-CONFIG_SPI_SH_MSIOF                             policy<{'riscv64': 'm'}>
 CONFIG_SPI_SIFIVE                               policy<{'riscv64': 'y'}>
 CONFIG_SPI_SLAVE                                policy<{'riscv64': 'y'}>
 CONFIG_SPI_SLAVE_SYSTEM_CONTROL                 policy<{'riscv64': 'm'}>
 CONFIG_SPI_SLAVE_TIME                           policy<{'riscv64': 'm'}>
 CONFIG_SPI_SN_F_OSPI                            policy<{'riscv64': 'm'}>
 CONFIG_SPI_SPIDEV                               policy<{'riscv64': 'm'}>
-CONFIG_SPI_SUN4I                                policy<{'riscv64': 'n'}>
-CONFIG_SPI_SUN6I                                policy<{'riscv64': 'm'}>
 CONFIG_SPI_TLE62X0                              policy<{'riscv64': 'm'}>
 CONFIG_SPI_XCOMM                                policy<{'riscv64': 'm'}>
 CONFIG_SPI_XILINX                               policy<{'riscv64': 'm'}>
@@ -8448,7 +8288,6 @@ CONFIG_STAGING_BOARD                            policy<{'riscv64': 'n'}>
 CONFIG_STAGING_MEDIA                            policy<{'riscv64': 'y'}>
 CONFIG_STAGING_MEDIA_DEPRECATED                 policy<{'riscv64': 'n'}>
 CONFIG_STANDALONE                               policy<{'riscv64': 'y'}>
-CONFIG_STARFIVE_WATCHDOG                        policy<{'riscv64': 'y'}>
 CONFIG_STATIC_KEYS_SELFTEST                     policy<{'riscv64': 'n'}>
 CONFIG_STATIC_USERMODEHELPER                    policy<{'riscv64': 'n'}>
 CONFIG_STE10XP                                  policy<{'riscv64': 'm'}>
@@ -8481,19 +8320,6 @@ CONFIG_STRIP_ASM_SYMS                           policy<{'riscv64': 'n'}>
 CONFIG_ST_UVIS25                                policy<{'riscv64': 'm'}>
 CONFIG_ST_UVIS25_I2C                            policy<{'riscv64': 'm'}>
 CONFIG_ST_UVIS25_SPI                            policy<{'riscv64': 'm'}>
-CONFIG_SUN20I_D1_CCU                            policy<{'riscv64': 'y'}>
-CONFIG_SUN20I_D1_R_CCU                          policy<{'riscv64': 'y'}>
-CONFIG_SUN20I_GPADC                             policy<{'riscv64': 'm'}>
-CONFIG_SUN20I_PPU                               policy<{'riscv64': 'y'}>
-CONFIG_SUN4I_EMAC                               policy<{'riscv64': 'n'}>
-CONFIG_SUN4I_GPADC                              policy<{'riscv64': 'm'}>
-CONFIG_SUN4I_TIMER                              policy<{'riscv64': 'y'}>
-CONFIG_SUN50I_DE2_BUS                           policy<{'riscv64': 'n'}>
-CONFIG_SUN50I_IOMMU                             policy<{'riscv64': 'y'}>
-CONFIG_SUN6I_MSGBOX                             policy<{'riscv64': 'm'}>
-CONFIG_SUN6I_RTC_CCU                            policy<{'riscv64': 'y'}>
-CONFIG_SUN8I_DE2_CCU                            policy<{'riscv64': 'y'}>
-CONFIG_SUN8I_THERMAL                            policy<{'riscv64': 'm'}>
 CONFIG_SUNDANCE                                 policy<{'riscv64': 'm'}>
 CONFIG_SUNDANCE_MMIO                            policy<{'riscv64': 'n'}>
 CONFIG_SUNGEM                                   policy<{'riscv64': 'm'}>
@@ -8504,18 +8330,12 @@ CONFIG_SUNRPC_DEBUG                             policy<{'riscv64': 'y'}>
 CONFIG_SUNRPC_GSS                               policy<{'riscv64': 'm'}>
 CONFIG_SUNRPC_SWAP                              policy<{'riscv64': 'y'}>
 CONFIG_SUNRPC_XPRT_RDMA                         policy<{'riscv64': 'm'}>
-CONFIG_SUNXI_CCU                                policy<{'riscv64': 'y'}>
-CONFIG_SUNXI_RSB                                policy<{'riscv64': 'm'}>
-CONFIG_SUNXI_SRAM                               policy<{'riscv64': 'y'}>
-CONFIG_SUNXI_WATCHDOG                           policy<{'riscv64': 'y'}>
 CONFIG_SUN_PARTITION                            policy<{'riscv64': 'y'}>
 CONFIG_SUSPEND                                  policy<{'riscv64': 'y'}>
 CONFIG_SUSPEND_FREEZER                          policy<{'riscv64': 'y'}>
 CONFIG_SUSPEND_SKIP_SYNC                        policy<{'riscv64': 'n'}>
 CONFIG_SVC_I3C_MASTER                           policy<{'riscv64': 'm'}>
 CONFIG_SWAP                                     policy<{'riscv64': 'y'}>
-CONFIG_SWIOTLB                                  policy<{'riscv64': 'y'}>
-CONFIG_SWIOTLB_DYNAMIC                          policy<{'riscv64': 'y'}>
 CONFIG_SWPHY                                    policy<{'riscv64': 'y'}>
 CONFIG_SW_SYNC                                  policy<{'riscv64': 'y'}>
 CONFIG_SX9310                                   policy<{'riscv64': 'm'}>
@@ -8810,7 +8630,6 @@ CONFIG_TOUCHSCREEN_SIS_I2C                      policy<{'riscv64': 'm'}>
 CONFIG_TOUCHSCREEN_ST1232                       policy<{'riscv64': 'm'}>
 CONFIG_TOUCHSCREEN_STMFTS                       policy<{'riscv64': 'm'}>
 CONFIG_TOUCHSCREEN_STMPE                        policy<{'riscv64': 'm'}>
-CONFIG_TOUCHSCREEN_SUN4I                        policy<{'riscv64': 'n'}>
 CONFIG_TOUCHSCREEN_SUR40                        policy<{'riscv64': 'm'}>
 CONFIG_TOUCHSCREEN_SURFACE3_SPI                 policy<{'riscv64': 'm'}>
 CONFIG_TOUCHSCREEN_SX8654                       policy<{'riscv64': 'm'}>
@@ -8904,10 +8723,11 @@ CONFIG_TWL4030_WATCHDOG                         policy<{'riscv64': 'm'}>
 CONFIG_TWL6030_GPADC                            policy<{'riscv64': 'm'}>
 CONFIG_TWL6040_CORE                             policy<{'riscv64': 'y'}>
 CONFIG_TXGBE                                    policy<{'riscv64': 'm'}>
-CONFIG_TYPEC                                    policy<{'riscv64': 'm'}>
+CONFIG_TYPEC                                    policy<{'riscv64': 'y'}>
 CONFIG_TYPEC_ANX7411                            policy<{'riscv64': 'm'}>
 CONFIG_TYPEC_DP_ALTMODE                         policy<{'riscv64': 'm'}>
 CONFIG_TYPEC_FUSB302                            policy<{'riscv64': 'm'}>
+CONFIG_TYPEC_FUSB303B                           policy<{'riscv64': 'y'}>
 CONFIG_TYPEC_HD3SS3220                          policy<{'riscv64': 'm'}>
 CONFIG_TYPEC_MT6360                             policy<{'riscv64': 'm'}>
 CONFIG_TYPEC_MUX_FSA4480                        policy<{'riscv64': 'm'}>
@@ -8921,7 +8741,7 @@ CONFIG_TYPEC_STUSB160X                          policy<{'riscv64': 'm'}>
 CONFIG_TYPEC_TCPCI                              policy<{'riscv64': 'm'}>
 CONFIG_TYPEC_TCPCI_MAXIM                        policy<{'riscv64': 'm'}>
 CONFIG_TYPEC_TCPCI_MT6370                       policy<{'riscv64': 'm'}>
-CONFIG_TYPEC_TCPM                               policy<{'riscv64': 'm'}>
+CONFIG_TYPEC_TCPM                               policy<{'riscv64': 'y'}>
 CONFIG_TYPEC_TPS6598X                           policy<{'riscv64': 'm'}>
 CONFIG_TYPEC_UCSI                               policy<{'riscv64': 'm'}>
 CONFIG_TYPEC_WUSB3801                           policy<{'riscv64': 'm'}>
@@ -9013,7 +8833,6 @@ CONFIG_USB_CDNS3                                policy<{'riscv64': 'm'}>
 CONFIG_USB_CDNS3_GADGET                         policy<{'riscv64': 'y'}>
 CONFIG_USB_CDNS3_HOST                           policy<{'riscv64': 'y'}>
 CONFIG_USB_CDNS3_PCI_WRAP                       policy<{'riscv64': 'm'}>
-CONFIG_USB_CDNS3_STARFIVE                       policy<{'riscv64': 'm'}>
 CONFIG_USB_CDNSP_GADGET                         policy<{'riscv64': 'y'}>
 CONFIG_USB_CDNSP_HOST                           policy<{'riscv64': 'y'}>
 CONFIG_USB_CDNSP_PCI                            policy<{'riscv64': 'm'}>
@@ -9063,14 +8882,12 @@ CONFIG_USB_DWC2_DEBUG                           policy<{'riscv64': 'n'}>
 CONFIG_USB_DWC2_HOST                            policy<{'riscv64': 'y'}>
 CONFIG_USB_DWC2_PCI                             policy<{'riscv64': 'm'}>
 CONFIG_USB_DWC2_TRACK_MISSED_SOFS               policy<{'riscv64': 'n'}>
-CONFIG_USB_DWC3                                 policy<{'riscv64': 'm'}>
-CONFIG_USB_DWC3_DUAL_ROLE                       policy<{'riscv64': 'y'}>
-CONFIG_USB_DWC3_GADGET                          policy<{'riscv64': 'n'}>
+CONFIG_USB_DWC3                                 policy<{'riscv64': 'y'}>
+CONFIG_USB_DWC3_ESWIN                           policy<{'riscv64': 'y'}>
 CONFIG_USB_DWC3_HAPS                            policy<{'riscv64': 'm'}>
-CONFIG_USB_DWC3_HOST                            policy<{'riscv64': 'n'}>
-CONFIG_USB_DWC3_OF_SIMPLE                       policy<{'riscv64': 'm'}>
+CONFIG_USB_DWC3_HOST                            policy<{'riscv64': 'y'}>
+CONFIG_USB_DWC3_OF_SIMPLE                       policy<{'riscv64': 'n'}>
 CONFIG_USB_DWC3_PCI                             policy<{'riscv64': 'm'}>
-CONFIG_USB_DWC3_ULPI                            policy<{'riscv64': 'y'}>
 CONFIG_USB_DYNAMIC_MINORS                       policy<{'riscv64': 'y'}>
 CONFIG_USB_EG20T                                policy<{'riscv64': 'm'}>
 CONFIG_USB_EHCI_FSL                             policy<{'riscv64': 'm'}>
@@ -9080,7 +8897,6 @@ CONFIG_USB_EHCI_TT_NEWSCHED                     policy<{'riscv64': 'y'}>
 CONFIG_USB_EHSET_TEST_FIXTURE                   policy<{'riscv64': 'm'}>
 CONFIG_USB_EMI26                                policy<{'riscv64': 'm'}>
 CONFIG_USB_EMI62                                policy<{'riscv64': 'm'}>
-CONFIG_USB_EMXX                                 policy<{'riscv64': 'm'}>
 CONFIG_USB_EPSON2888                            policy<{'riscv64': 'y'}>
 CONFIG_USB_ETH                                  policy<{'riscv64': 'm'}>
 CONFIG_USB_ETH_EEM                              policy<{'riscv64': 'y'}>
@@ -9230,8 +9046,6 @@ CONFIG_USB_MSI2500                              policy<{'riscv64': 'm'}>
 CONFIG_USB_MUSB_DUAL_ROLE                       policy<{'riscv64': 'y'}>
 CONFIG_USB_MUSB_GADGET                          policy<{'riscv64': 'n'}>
 CONFIG_USB_MUSB_HOST                            policy<{'riscv64': 'n'}>
-CONFIG_USB_MUSB_POLARFIRE_SOC                   policy<{'riscv64': 'm'}>
-CONFIG_USB_MUSB_SUNXI                           policy<{'riscv64': 'm'}>
 CONFIG_USB_MV_U3D                               policy<{'riscv64': 'm'}>
 CONFIG_USB_MV_UDC                               policy<{'riscv64': 'm'}>
 CONFIG_USB_NET2272                              policy<{'riscv64': 'm'}>
@@ -9288,11 +9102,6 @@ CONFIG_USB_R8A66597_HCD                         policy<{'riscv64': 'm'}>
 CONFIG_USB_RAINSHADOW_CEC                       policy<{'riscv64': 'm'}>
 CONFIG_USB_RAREMONO                             policy<{'riscv64': 'm'}>
 CONFIG_USB_RAW_GADGET                           policy<{'riscv64': 'm'}>
-CONFIG_USB_RENESAS_USB3                         policy<{'riscv64': 'm'}>
-CONFIG_USB_RENESAS_USBF                         policy<{'riscv64': 'm'}>
-CONFIG_USB_RENESAS_USBHS                        policy<{'riscv64': 'm'}>
-CONFIG_USB_RENESAS_USBHS_HCD                    policy<{'riscv64': 'm'}>
-CONFIG_USB_RENESAS_USBHS_UDC                    policy<{'riscv64': 'm'}>
 CONFIG_USB_ROLE_SWITCH                          policy<{'riscv64': 'y'}>
 CONFIG_USB_RTL8150                              policy<{'riscv64': 'm'}>
 CONFIG_USB_RTL8152                              policy<{'riscv64': 'm'}>
@@ -9397,8 +9206,7 @@ CONFIG_USB_VL600                                policy<{'riscv64': 'm'}>
 CONFIG_USB_WDM                                  policy<{'riscv64': 'm'}>
 CONFIG_USB_XHCI_PCI                             policy<{'riscv64': 'm'}>
 CONFIG_USB_XHCI_PCI_RENESAS                     policy<{'riscv64': 'm'}>
-CONFIG_USB_XHCI_PLATFORM                        policy<{'riscv64': 'm'}>
-CONFIG_USB_XHCI_RCAR                            policy<{'riscv64': 'm'}>
+CONFIG_USB_XHCI_PLATFORM                        policy<{'riscv64': 'y'}>
 CONFIG_USB_XUSBATM                              policy<{'riscv64': 'm'}>
 CONFIG_USB_YUREX                                policy<{'riscv64': 'm'}>
 CONFIG_USB_ZD1201                               policy<{'riscv64': 'm'}>
@@ -9420,9 +9228,7 @@ CONFIG_V4L2_CCI                                 policy<{'riscv64': 'm'}>
 CONFIG_V4L2_CCI_I2C                             policy<{'riscv64': 'm'}>
 CONFIG_V4L2_FLASH_LED_CLASS                     policy<{'riscv64': 'm'}>
 CONFIG_V4L2_FWNODE                              policy<{'riscv64': 'm'}>
-CONFIG_V4L2_H264                                policy<{'riscv64': 'm'}>
 CONFIG_V4L2_MEM2MEM_DEV                         policy<{'riscv64': 'm'}>
-CONFIG_V4L2_VP9                                 policy<{'riscv64': 'm'}>
 CONFIG_V4L_MEM2MEM_DRIVERS                      policy<{'riscv64': 'y'}>
 CONFIG_V4L_PLATFORM_DRIVERS                     policy<{'riscv64': 'y'}>
 CONFIG_V4L_TEST_DRIVERS                         policy<{'riscv64': 'y'}>
@@ -9550,8 +9356,6 @@ CONFIG_VIDEO_GO7007_LOADER                      policy<{'riscv64': 'm'}>
 CONFIG_VIDEO_GO7007_USB                         policy<{'riscv64': 'm'}>
 CONFIG_VIDEO_GO7007_USB_S2250_BOARD             policy<{'riscv64': 'm'}>
 CONFIG_VIDEO_GS1662                             policy<{'riscv64': 'm'}>
-CONFIG_VIDEO_HANTRO                             policy<{'riscv64': 'm'}>
-CONFIG_VIDEO_HANTRO_SUNXI                       policy<{'riscv64': 'y'}>
 CONFIG_VIDEO_HDPVR                              policy<{'riscv64': 'm'}>
 CONFIG_VIDEO_HEXIUM_GEMINI                      policy<{'riscv64': 'm'}>
 CONFIG_VIDEO_HEXIUM_ORION                       policy<{'riscv64': 'm'}>
@@ -9635,19 +9439,9 @@ CONFIG_VIDEO_PVRUSB2                            policy<{'riscv64': 'm'}>
 CONFIG_VIDEO_PVRUSB2_DEBUGIFC                   policy<{'riscv64': 'n'}>
 CONFIG_VIDEO_PVRUSB2_DVB                        policy<{'riscv64': 'y'}>
 CONFIG_VIDEO_PVRUSB2_SYSFS                      policy<{'riscv64': 'y'}>
-CONFIG_VIDEO_RCAR_CSI2                          policy<{'riscv64': 'm'}>
-CONFIG_VIDEO_RCAR_DRIF                          policy<{'riscv64': 'm'}>
-CONFIG_VIDEO_RCAR_ISP                           policy<{'riscv64': 'm'}>
-CONFIG_VIDEO_RCAR_VIN                           policy<{'riscv64': 'm'}>
 CONFIG_VIDEO_RDACM20                            policy<{'riscv64': 'm'}>
 CONFIG_VIDEO_RDACM21                            policy<{'riscv64': 'm'}>
-CONFIG_VIDEO_RENESAS_FCP                        policy<{'riscv64': 'm'}>
-CONFIG_VIDEO_RENESAS_FDP1                       policy<{'riscv64': 'm'}>
-CONFIG_VIDEO_RENESAS_JPU                        policy<{'riscv64': 'm'}>
-CONFIG_VIDEO_RENESAS_VSP1                       policy<{'riscv64': 'm'}>
 CONFIG_VIDEO_RJ54N1                             policy<{'riscv64': 'm'}>
-CONFIG_VIDEO_RZG2L_CRU                          policy<{'riscv64': 'm'}>
-CONFIG_VIDEO_RZG2L_CSI2                         policy<{'riscv64': 'm'}>
 CONFIG_VIDEO_S5C73M3                            policy<{'riscv64': 'm'}>
 CONFIG_VIDEO_S5K5BAF                            policy<{'riscv64': 'm'}>
 CONFIG_VIDEO_S5K6A3                             policy<{'riscv64': 'm'}>
@@ -9671,15 +9465,6 @@ CONFIG_VIDEO_SONY_BTF_MPX                       policy<{'riscv64': 'm'}>
 CONFIG_VIDEO_STK1160                            policy<{'riscv64': 'm'}>
 CONFIG_VIDEO_ST_MIPID02                         policy<{'riscv64': 'm'}>
 CONFIG_VIDEO_ST_VGXY61                          policy<{'riscv64': 'm'}>
-CONFIG_VIDEO_SUN4I_CSI                          policy<{'riscv64': 'm'}>
-CONFIG_VIDEO_SUN6I_CSI                          policy<{'riscv64': 'm'}>
-CONFIG_VIDEO_SUN6I_ISP                          policy<{'riscv64': 'm'}>
-CONFIG_VIDEO_SUN6I_MIPI_CSI2                    policy<{'riscv64': 'm'}>
-CONFIG_VIDEO_SUN8I_A83T_MIPI_CSI2               policy<{'riscv64': 'm'}>
-CONFIG_VIDEO_SUN8I_DEINTERLACE                  policy<{'riscv64': 'm'}>
-CONFIG_VIDEO_SUN8I_ROTATE                       policy<{'riscv64': 'm'}>
-CONFIG_VIDEO_SUNXI                              policy<{'riscv64': 'y'}>
-CONFIG_VIDEO_SUNXI_CEDRUS                       policy<{'riscv64': 'm'}>
 CONFIG_VIDEO_TC358743                           policy<{'riscv64': 'm'}>
 CONFIG_VIDEO_TC358743_CEC                       policy<{'riscv64': 'y'}>
 CONFIG_VIDEO_TC358746                           policy<{'riscv64': 'm'}>
-- 
2.47.0

